<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8"/>
    <title>Test Report</title>
    <style>body {
	font-family: Helvetica, Arial, sans-serif;
	font-size: 12px;
	min-width: 1200px;
	color: #999;
}

h1 {
	font-size: 24px;
	color: black;
}

h2 {
	font-size: 16px;
	color: black;
}

p {
    color: black;
}

a {
	color: #999;
}

table {
	border-collapse: collapse;
}

/******************************
 * SUMMARY INFORMATION
 ******************************/

#environment td {
	padding: 5px;
	border: 1px solid #E6E6E6;
}

#environment tr:nth-child(odd) {
	background-color: #f6f6f6;
}

/******************************
 * TEST RESULT COLORS
 ******************************/
span.passed, .passed .col-result {
	color: green;
}
span.skipped, span.xfailed, span.rerun, .skipped .col-result, .xfailed .col-result, .rerun .col-result {
	color: orange;
}
span.error, span.failed, span.xpassed, .error .col-result, .failed .col-result, .xpassed .col-result  {
	color: red;
}


/******************************
 * RESULTS TABLE
 *
 * 1. Table Layout
 * 2. Extra
 * 3. Sorting items
 *
 ******************************/

/*------------------
 * 1. Table Layout
 *------------------*/

#results-table {
	border: 1px solid #e6e6e6;
	color: #999;
	font-size: 12px;
	width: 100%
}

#results-table th, #results-table td {
	padding: 5px;
	border: 1px solid #E6E6E6;
	text-align: left
}
#results-table th {
	font-weight: bold
}

/*------------------
 * 2. Extra
 *------------------*/

.log:only-child {
	height: inherit
}
.log {
	background-color: #e6e6e6;
	border: 1px solid #e6e6e6;
	color: black;
	display: block;
	font-family: "Courier New", Courier, monospace;
	height: 230px;
	overflow-y: scroll;
	padding: 5px;
	white-space: pre-wrap
}
div.image {
	border: 1px solid #e6e6e6;
	float: right;
	height: 240px;
	margin-left: 5px;
	overflow: hidden;
	width: 320px
}
div.image img {
	width: 320px
}
.collapsed {
	display: none;
}
.expander::after {
	content: " (show details)";
	color: #BBB;
	font-style: italic;
	cursor: pointer;
}
.collapser::after {
	content: " (hide details)";
	color: #BBB;
	font-style: italic;
	cursor: pointer;
}

/*------------------
 * 3. Sorting items
 *------------------*/
.sortable {
	cursor: pointer;
}

.sort-icon {
	font-size: 0px;
	float: left;
	margin-right: 5px;
	margin-top: 5px;
	/*triangle*/
	width: 0;
	height: 0;
	border-left: 8px solid transparent;
	border-right: 8px solid transparent;
}

.inactive .sort-icon {
	/*finish triangle*/
	border-top: 8px solid #E6E6E6;
}

.asc.active .sort-icon {
	/*finish triangle*/
	border-bottom: 8px solid #999;
}

.desc.active .sort-icon {
	/*finish triangle*/
	border-top: 8px solid #999;
}
</style></head>
  <body onLoad="init()">
    <script>/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this file,
 * You can obtain one at http://mozilla.org/MPL/2.0/. */


function toArray(iter) {
    if (iter === null) {
        return null;
    }
    return Array.prototype.slice.call(iter);
}

function find(selector, elem) {
    if (!elem) {
        elem = document;
    }
    return elem.querySelector(selector);
}

function find_all(selector, elem) {
    if (!elem) {
        elem = document;
    }
    return toArray(elem.querySelectorAll(selector));
}

function sort_column(elem) {
    toggle_sort_states(elem);
    var colIndex = toArray(elem.parentNode.childNodes).indexOf(elem);
    var key;
    if (elem.classList.contains('numeric')) {
        key = key_num;
    } else if (elem.classList.contains('result')) {
        key = key_result;
    } else {
        key = key_alpha;
    }
    sort_table(elem, key(colIndex));
}

function show_all_extras() {
    find_all('.col-result').forEach(show_extras);
}

function hide_all_extras() {
    find_all('.col-result').forEach(hide_extras);
}

function show_extras(colresult_elem) {
    var extras = colresult_elem.parentNode.nextElementSibling;
    var expandcollapse = colresult_elem.firstElementChild;
    extras.classList.remove("collapsed");
    expandcollapse.classList.remove("expander");
    expandcollapse.classList.add("collapser");
}

function hide_extras(colresult_elem) {
    var extras = colresult_elem.parentNode.nextElementSibling;
    var expandcollapse = colresult_elem.firstElementChild;
    extras.classList.add("collapsed");
    expandcollapse.classList.remove("collapser");
    expandcollapse.classList.add("expander");
}

function show_filters() {
    var filter_items = document.getElementsByClassName('filter');
    for (var i = 0; i < filter_items.length; i++)
        filter_items[i].hidden = false;
}

function add_collapse() {
    // Add links for show/hide all
    var resulttable = find('table#results-table');
    var showhideall = document.createElement("p");
    showhideall.innerHTML = '<a href="javascript:show_all_extras()">Show all details</a> / ' +
                            '<a href="javascript:hide_all_extras()">Hide all details</a>';
    resulttable.parentElement.insertBefore(showhideall, resulttable);

    // Add show/hide link to each result
    find_all('.col-result').forEach(function(elem) {
        var collapsed = get_query_parameter('collapsed') || 'Passed';
        var extras = elem.parentNode.nextElementSibling;
        var expandcollapse = document.createElement("span");
        if (collapsed.includes(elem.innerHTML)) {
            extras.classList.add("collapsed");
            expandcollapse.classList.add("expander");
        } else {
            expandcollapse.classList.add("collapser");
        }
        elem.appendChild(expandcollapse);

        elem.addEventListener("click", function(event) {
            if (event.currentTarget.parentNode.nextElementSibling.classList.contains("collapsed")) {
                show_extras(event.currentTarget);
            } else {
                hide_extras(event.currentTarget);
            }
        });
    })
}

function get_query_parameter(name) {
    var match = RegExp('[?&]' + name + '=([^&]*)').exec(window.location.search);
    return match && decodeURIComponent(match[1].replace(/\+/g, ' '));
}

function init () {
    reset_sort_headers();

    add_collapse();

    show_filters();

    toggle_sort_states(find('.initial-sort'));

    find_all('.sortable').forEach(function(elem) {
        elem.addEventListener("click",
                              function(event) {
                                  sort_column(elem);
                              }, false)
    });

};

function sort_table(clicked, key_func) {
    var rows = find_all('.results-table-row');
    var reversed = !clicked.classList.contains('asc');
    var sorted_rows = sort(rows, key_func, reversed);
    /* Whole table is removed here because browsers acts much slower
     * when appending existing elements.
     */
    var thead = document.getElementById("results-table-head");
    document.getElementById('results-table').remove();
    var parent = document.createElement("table");
    parent.id = "results-table";
    parent.appendChild(thead);
    sorted_rows.forEach(function(elem) {
        parent.appendChild(elem);
    });
    document.getElementsByTagName("BODY")[0].appendChild(parent);
}

function sort(items, key_func, reversed) {
    var sort_array = items.map(function(item, i) {
        return [key_func(item), i];
    });
    var multiplier = reversed ? -1 : 1;

    sort_array.sort(function(a, b) {
        var key_a = a[0];
        var key_b = b[0];
        return multiplier * (key_a >= key_b ? 1 : -1);
    });

    return sort_array.map(function(item) {
        var index = item[1];
        return items[index];
    });
}

function key_alpha(col_index) {
    return function(elem) {
        return elem.childNodes[1].childNodes[col_index].firstChild.data.toLowerCase();
    };
}

function key_num(col_index) {
    return function(elem) {
        return parseFloat(elem.childNodes[1].childNodes[col_index].firstChild.data);
    };
}

function key_result(col_index) {
    return function(elem) {
        var strings = ['Error', 'Failed', 'Rerun', 'XFailed', 'XPassed',
                       'Skipped', 'Passed'];
        return strings.indexOf(elem.childNodes[1].childNodes[col_index].firstChild.data);
    };
}

function reset_sort_headers() {
    find_all('.sort-icon').forEach(function(elem) {
        elem.parentNode.removeChild(elem);
    });
    find_all('.sortable').forEach(function(elem) {
        var icon = document.createElement("div");
        icon.className = "sort-icon";
        icon.textContent = "vvv";
        elem.insertBefore(icon, elem.firstChild);
        elem.classList.remove("desc", "active");
        elem.classList.add("asc", "inactive");
    });
}

function toggle_sort_states(elem) {
    //if active, toggle between asc and desc
    if (elem.classList.contains('active')) {
        elem.classList.toggle('asc');
        elem.classList.toggle('desc');
    }

    //if inactive, reset all other functions and add ascending active
    if (elem.classList.contains('inactive')) {
        reset_sort_headers();
        elem.classList.remove('inactive');
        elem.classList.add('active');
    }
}

function is_all_rows_hidden(value) {
  return value.hidden == false;
}

function filter_table(elem) {
    var outcome_att = "data-test-result";
    var outcome = elem.getAttribute(outcome_att);
    class_outcome = outcome + " results-table-row";
    var outcome_rows = document.getElementsByClassName(class_outcome);

    for(var i = 0; i < outcome_rows.length; i++){
        outcome_rows[i].hidden = !elem.checked;
    }

    var rows = find_all('.results-table-row').filter(is_all_rows_hidden);
    var all_rows_hidden = rows.length == 0 ? true : false;
    var not_found_message = document.getElementById("not-found-message");
    not_found_message.hidden = !all_rows_hidden;
}
</script>
    <h1>locobot.html</h1>
    <p>Report generated on 26-Jun-2019 at 21:43:30 by <a href="https://pypi.python.org/pypi/pytest-html">pytest-html</a> v1.20.0</p>
    <h2>Environment</h2>
    <table id="environment">
      <tr>
        <td>Packages</td>
        <td>{&apos;py&apos;: &apos;1.8.0&apos;, &apos;pytest&apos;: &apos;4.6.3&apos;, &apos;pluggy&apos;: &apos;0.12.0&apos;}</td></tr>
      <tr>
        <td>Platform</td>
        <td>Linux-4.15.0-52-generic-x86_64-with-Ubuntu-16.04-xenial</td></tr>
      <tr>
        <td>Plugins</td>
        <td>{u&apos;cov&apos;: u&apos;2.6.1&apos;, u&apos;html&apos;: u&apos;1.20.0&apos;, u&apos;metadata&apos;: u&apos;1.8.0&apos;}</td></tr>
      <tr>
        <td>Python</td>
        <td>2.7.12</td></tr></table>
    <h2>Summary</h2>
    <p>9 tests ran in 26.94 seconds. </p>
    <p class="filter" hidden="true">(Un)check the boxes to filter the results.</p><input checked="true" class="filter" data-test-result="passed" disabled="true" hidden="true" name="filter_checkbox" onChange="filter_table(this)" type="checkbox"/><span class="passed">0 passed</span>, <input checked="true" class="filter" data-test-result="skipped" disabled="true" hidden="true" name="filter_checkbox" onChange="filter_table(this)" type="checkbox"/><span class="skipped">0 skipped</span>, <input checked="true" class="filter" data-test-result="failed" hidden="true" name="filter_checkbox" onChange="filter_table(this)" type="checkbox"/><span class="failed">9 failed</span>, <input checked="true" class="filter" data-test-result="error" hidden="true" name="filter_checkbox" onChange="filter_table(this)" type="checkbox"/><span class="error">26 errors</span>, <input checked="true" class="filter" data-test-result="xfailed" disabled="true" hidden="true" name="filter_checkbox" onChange="filter_table(this)" type="checkbox"/><span class="xfailed">0 expected failures</span>, <input checked="true" class="filter" data-test-result="xpassed" disabled="true" hidden="true" name="filter_checkbox" onChange="filter_table(this)" type="checkbox"/><span class="xpassed">0 unexpected passes</span>
    <h2>Results</h2>
    <table id="results-table">
      <thead id="results-table-head">
        <tr>
          <th class="sortable result initial-sort" col="result">Result</th>
          <th class="sortable" col="name">Test</th>
          <th class="sortable numeric" col="duration">Duration</th>
          <th>Links</th></tr>
        <tr hidden="true" id="not-found-message">
          <th colspan="4">No results found. Try to check the filters</th></tr></thead>
      <tbody class="error results-table-row">
        <tr>
          <td class="col-result">Error</td>
          <td class="col-name">tests/test_arm_utils.py::test_get_ee_pose::setup</td>
          <td class="col-duration">0.00</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">@pytest.fixture(scope="module")<br/>    def create_robot():<br/>&gt;       return Robot('locobot', use_base=False)<br/><br/>test_arm_utils.py:15: <br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/core.py:105: in __init__<br/>    self.arm = arm_class(self.configs, **arm_config)<br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/locobot/arm.py:40: in __init__<br/>    use_arm = rospy.get_param('use_arm', False)<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/client.py:465: in get_param<br/>    return _param_server[param_name] #MasterProxy does all the magic for us<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/msproxy.py:121: in __getitem__<br/>    code, msg, value = self.target.getParam(rospy.names.get_caller_id(), resolved_key)<br/>/usr/lib/python2.7/xmlrpclib.py:1243: in __call__<br/>    return self.__send(self.__name, args)<br/>/usr/lib/python2.7/xmlrpclib.py:1602: in __request<br/>    verbose=self.__verbose<br/>/usr/lib/python2.7/xmlrpclib.py:1283: in request<br/>    return self.single_request(host, handler, request_body, verbose)<br/>/usr/lib/python2.7/xmlrpclib.py:1311: in single_request<br/>    self.send_content(h, request_body)<br/>/usr/lib/python2.7/xmlrpclib.py:1459: in send_content<br/>    connection.endheaders(request_body)<br/>/usr/lib/python2.7/httplib.py:1053: in endheaders<br/>    self._send_output(message_body)<br/>/usr/lib/python2.7/httplib.py:897: in _send_output<br/>    self.send(msg)<br/>/usr/lib/python2.7/httplib.py:859: in send<br/>    self.connect()<br/>/usr/lib/python2.7/httplib.py:836: in connect<br/>    self.timeout, self.source_address)<br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/><br/>address = ('localhost', 11311), timeout = &lt;object object at 0x7f5b8b5b83b0&gt;, source_address = None<br/><br/>    def create_connection(address, timeout=_GLOBAL_DEFAULT_TIMEOUT,<br/>                          source_address=None):<br/>        """Connect to *address* and return the socket object.<br/>    <br/>        Convenience function.  Connect to *address* (a 2-tuple ``(host,<br/>        port)``) and return the socket object.  Passing the optional<br/>        *timeout* parameter will set the timeout on the socket instance<br/>        before attempting to connect.  If no *timeout* is supplied, the<br/>        global default timeout setting returned by :func:`getdefaulttimeout`<br/>        is used.  If *source_address* is set it must be a tuple of (host, port)<br/>        for the socket to bind as a source address before making the connection.<br/>        A host of '' or port 0 tells the OS to use the default.<br/>        """<br/>    <br/>        host, port = address<br/>        err = None<br/>        for res in getaddrinfo(host, port, 0, SOCK_STREAM):<br/>            af, socktype, proto, canonname, sa = res<br/>            sock = None<br/>            try:<br/>                sock = socket(af, socktype, proto)<br/>                if timeout is not _GLOBAL_DEFAULT_TIMEOUT:<br/>                    sock.settimeout(timeout)<br/>                if source_address:<br/>                    sock.bind(source_address)<br/>                sock.connect(sa)<br/>                return sock<br/>    <br/>            except error as _:<br/>                err = _<br/>                if sock is not None:<br/>                    sock.close()<br/>    <br/>        if err is not None:<br/>&gt;           raise err<br/><span class="error">E           error: [Errno 111] Connection refused</span><br/><br/>/usr/lib/python2.7/socket.py:575: error<br/>---------------------------- Captured stdout setup -----------------------------<br/>['husky', 'locobot', 'sawyer', 'turtlebot']
</div></td></tr></tbody>
      <tbody class="error results-table-row">
        <tr>
          <td class="col-result">Error</td>
          <td class="col-name">tests/test_arm_utils.py::test_get_jacobian::setup</td>
          <td class="col-duration">0.00</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">@pytest.fixture(scope="module")<br/>    def create_robot():<br/>&gt;       return Robot('locobot', use_base=False)<br/><br/>test_arm_utils.py:15: <br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/core.py:105: in __init__<br/>    self.arm = arm_class(self.configs, **arm_config)<br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/locobot/arm.py:40: in __init__<br/>    use_arm = rospy.get_param('use_arm', False)<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/client.py:465: in get_param<br/>    return _param_server[param_name] #MasterProxy does all the magic for us<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/msproxy.py:121: in __getitem__<br/>    code, msg, value = self.target.getParam(rospy.names.get_caller_id(), resolved_key)<br/>/usr/lib/python2.7/xmlrpclib.py:1243: in __call__<br/>    return self.__send(self.__name, args)<br/>/usr/lib/python2.7/xmlrpclib.py:1602: in __request<br/>    verbose=self.__verbose<br/>/usr/lib/python2.7/xmlrpclib.py:1283: in request<br/>    return self.single_request(host, handler, request_body, verbose)<br/>/usr/lib/python2.7/xmlrpclib.py:1311: in single_request<br/>    self.send_content(h, request_body)<br/>/usr/lib/python2.7/xmlrpclib.py:1459: in send_content<br/>    connection.endheaders(request_body)<br/>/usr/lib/python2.7/httplib.py:1053: in endheaders<br/>    self._send_output(message_body)<br/>/usr/lib/python2.7/httplib.py:897: in _send_output<br/>    self.send(msg)<br/>/usr/lib/python2.7/httplib.py:859: in send<br/>    self.connect()<br/>/usr/lib/python2.7/httplib.py:836: in connect<br/>    self.timeout, self.source_address)<br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/><br/>address = ('localhost', 11311), timeout = &lt;object object at 0x7f5b8b5b83b0&gt;, source_address = None<br/><br/>    def create_connection(address, timeout=_GLOBAL_DEFAULT_TIMEOUT,<br/>                          source_address=None):<br/>        """Connect to *address* and return the socket object.<br/>    <br/>        Convenience function.  Connect to *address* (a 2-tuple ``(host,<br/>        port)``) and return the socket object.  Passing the optional<br/>        *timeout* parameter will set the timeout on the socket instance<br/>        before attempting to connect.  If no *timeout* is supplied, the<br/>        global default timeout setting returned by :func:`getdefaulttimeout`<br/>        is used.  If *source_address* is set it must be a tuple of (host, port)<br/>        for the socket to bind as a source address before making the connection.<br/>        A host of '' or port 0 tells the OS to use the default.<br/>        """<br/>    <br/>        host, port = address<br/>        err = None<br/>        for res in getaddrinfo(host, port, 0, SOCK_STREAM):<br/>            af, socktype, proto, canonname, sa = res<br/>            sock = None<br/>            try:<br/>                sock = socket(af, socktype, proto)<br/>                if timeout is not _GLOBAL_DEFAULT_TIMEOUT:<br/>                    sock.settimeout(timeout)<br/>                if source_address:<br/>                    sock.bind(source_address)<br/>                sock.connect(sa)<br/>                return sock<br/>    <br/>            except error as _:<br/>                err = _<br/>                if sock is not None:<br/>                    sock.close()<br/>    <br/>        if err is not None:<br/>&gt;           raise err<br/><span class="error">E           error: [Errno 111] Connection refused</span><br/><br/>/usr/lib/python2.7/socket.py:575: error<br/></div></td></tr></tbody>
      <tbody class="error results-table-row">
        <tr>
          <td class="col-result">Error</td>
          <td class="col-name">tests/test_arm_utils.py::test_compute_fk_position::setup</td>
          <td class="col-duration">0.00</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">@pytest.fixture(scope="module")<br/>    def create_robot():<br/>&gt;       return Robot('locobot', use_base=False)<br/><br/>test_arm_utils.py:15: <br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/core.py:105: in __init__<br/>    self.arm = arm_class(self.configs, **arm_config)<br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/locobot/arm.py:40: in __init__<br/>    use_arm = rospy.get_param('use_arm', False)<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/client.py:465: in get_param<br/>    return _param_server[param_name] #MasterProxy does all the magic for us<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/msproxy.py:121: in __getitem__<br/>    code, msg, value = self.target.getParam(rospy.names.get_caller_id(), resolved_key)<br/>/usr/lib/python2.7/xmlrpclib.py:1243: in __call__<br/>    return self.__send(self.__name, args)<br/>/usr/lib/python2.7/xmlrpclib.py:1602: in __request<br/>    verbose=self.__verbose<br/>/usr/lib/python2.7/xmlrpclib.py:1283: in request<br/>    return self.single_request(host, handler, request_body, verbose)<br/>/usr/lib/python2.7/xmlrpclib.py:1311: in single_request<br/>    self.send_content(h, request_body)<br/>/usr/lib/python2.7/xmlrpclib.py:1459: in send_content<br/>    connection.endheaders(request_body)<br/>/usr/lib/python2.7/httplib.py:1053: in endheaders<br/>    self._send_output(message_body)<br/>/usr/lib/python2.7/httplib.py:897: in _send_output<br/>    self.send(msg)<br/>/usr/lib/python2.7/httplib.py:859: in send<br/>    self.connect()<br/>/usr/lib/python2.7/httplib.py:836: in connect<br/>    self.timeout, self.source_address)<br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/><br/>address = ('localhost', 11311), timeout = &lt;object object at 0x7f5b8b5b83b0&gt;, source_address = None<br/><br/>    def create_connection(address, timeout=_GLOBAL_DEFAULT_TIMEOUT,<br/>                          source_address=None):<br/>        """Connect to *address* and return the socket object.<br/>    <br/>        Convenience function.  Connect to *address* (a 2-tuple ``(host,<br/>        port)``) and return the socket object.  Passing the optional<br/>        *timeout* parameter will set the timeout on the socket instance<br/>        before attempting to connect.  If no *timeout* is supplied, the<br/>        global default timeout setting returned by :func:`getdefaulttimeout`<br/>        is used.  If *source_address* is set it must be a tuple of (host, port)<br/>        for the socket to bind as a source address before making the connection.<br/>        A host of '' or port 0 tells the OS to use the default.<br/>        """<br/>    <br/>        host, port = address<br/>        err = None<br/>        for res in getaddrinfo(host, port, 0, SOCK_STREAM):<br/>            af, socktype, proto, canonname, sa = res<br/>            sock = None<br/>            try:<br/>                sock = socket(af, socktype, proto)<br/>                if timeout is not _GLOBAL_DEFAULT_TIMEOUT:<br/>                    sock.settimeout(timeout)<br/>                if source_address:<br/>                    sock.bind(source_address)<br/>                sock.connect(sa)<br/>                return sock<br/>    <br/>            except error as _:<br/>                err = _<br/>                if sock is not None:<br/>                    sock.close()<br/>    <br/>        if err is not None:<br/>&gt;           raise err<br/><span class="error">E           error: [Errno 111] Connection refused</span><br/><br/>/usr/lib/python2.7/socket.py:575: error<br/></div></td></tr></tbody>
      <tbody class="error results-table-row">
        <tr>
          <td class="col-result">Error</td>
          <td class="col-name">tests/test_arm_utils.py::test_compute_fk_velocity::setup</td>
          <td class="col-duration">0.00</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">@pytest.fixture(scope="module")<br/>    def create_robot():<br/>&gt;       return Robot('locobot', use_base=False)<br/><br/>test_arm_utils.py:15: <br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/core.py:105: in __init__<br/>    self.arm = arm_class(self.configs, **arm_config)<br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/locobot/arm.py:40: in __init__<br/>    use_arm = rospy.get_param('use_arm', False)<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/client.py:465: in get_param<br/>    return _param_server[param_name] #MasterProxy does all the magic for us<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/msproxy.py:121: in __getitem__<br/>    code, msg, value = self.target.getParam(rospy.names.get_caller_id(), resolved_key)<br/>/usr/lib/python2.7/xmlrpclib.py:1243: in __call__<br/>    return self.__send(self.__name, args)<br/>/usr/lib/python2.7/xmlrpclib.py:1602: in __request<br/>    verbose=self.__verbose<br/>/usr/lib/python2.7/xmlrpclib.py:1283: in request<br/>    return self.single_request(host, handler, request_body, verbose)<br/>/usr/lib/python2.7/xmlrpclib.py:1311: in single_request<br/>    self.send_content(h, request_body)<br/>/usr/lib/python2.7/xmlrpclib.py:1459: in send_content<br/>    connection.endheaders(request_body)<br/>/usr/lib/python2.7/httplib.py:1053: in endheaders<br/>    self._send_output(message_body)<br/>/usr/lib/python2.7/httplib.py:897: in _send_output<br/>    self.send(msg)<br/>/usr/lib/python2.7/httplib.py:859: in send<br/>    self.connect()<br/>/usr/lib/python2.7/httplib.py:836: in connect<br/>    self.timeout, self.source_address)<br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/><br/>address = ('localhost', 11311), timeout = &lt;object object at 0x7f5b8b5b83b0&gt;, source_address = None<br/><br/>    def create_connection(address, timeout=_GLOBAL_DEFAULT_TIMEOUT,<br/>                          source_address=None):<br/>        """Connect to *address* and return the socket object.<br/>    <br/>        Convenience function.  Connect to *address* (a 2-tuple ``(host,<br/>        port)``) and return the socket object.  Passing the optional<br/>        *timeout* parameter will set the timeout on the socket instance<br/>        before attempting to connect.  If no *timeout* is supplied, the<br/>        global default timeout setting returned by :func:`getdefaulttimeout`<br/>        is used.  If *source_address* is set it must be a tuple of (host, port)<br/>        for the socket to bind as a source address before making the connection.<br/>        A host of '' or port 0 tells the OS to use the default.<br/>        """<br/>    <br/>        host, port = address<br/>        err = None<br/>        for res in getaddrinfo(host, port, 0, SOCK_STREAM):<br/>            af, socktype, proto, canonname, sa = res<br/>            sock = None<br/>            try:<br/>                sock = socket(af, socktype, proto)<br/>                if timeout is not _GLOBAL_DEFAULT_TIMEOUT:<br/>                    sock.settimeout(timeout)<br/>                if source_address:<br/>                    sock.bind(source_address)<br/>                sock.connect(sa)<br/>                return sock<br/>    <br/>            except error as _:<br/>                err = _<br/>                if sock is not None:<br/>                    sock.close()<br/>    <br/>        if err is not None:<br/>&gt;           raise err<br/><span class="error">E           error: [Errno 111] Connection refused</span><br/><br/>/usr/lib/python2.7/socket.py:575: error<br/></div></td></tr></tbody>
      <tbody class="error results-table-row">
        <tr>
          <td class="col-result">Error</td>
          <td class="col-name">tests/test_arm_utils.py::test_compute_ik[True]::setup</td>
          <td class="col-duration">0.00</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">@pytest.fixture(scope="module")<br/>    def create_robot():<br/>&gt;       return Robot('locobot', use_base=False)<br/><br/>test_arm_utils.py:15: <br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/core.py:105: in __init__<br/>    self.arm = arm_class(self.configs, **arm_config)<br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/locobot/arm.py:40: in __init__<br/>    use_arm = rospy.get_param('use_arm', False)<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/client.py:465: in get_param<br/>    return _param_server[param_name] #MasterProxy does all the magic for us<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/msproxy.py:121: in __getitem__<br/>    code, msg, value = self.target.getParam(rospy.names.get_caller_id(), resolved_key)<br/>/usr/lib/python2.7/xmlrpclib.py:1243: in __call__<br/>    return self.__send(self.__name, args)<br/>/usr/lib/python2.7/xmlrpclib.py:1602: in __request<br/>    verbose=self.__verbose<br/>/usr/lib/python2.7/xmlrpclib.py:1283: in request<br/>    return self.single_request(host, handler, request_body, verbose)<br/>/usr/lib/python2.7/xmlrpclib.py:1311: in single_request<br/>    self.send_content(h, request_body)<br/>/usr/lib/python2.7/xmlrpclib.py:1459: in send_content<br/>    connection.endheaders(request_body)<br/>/usr/lib/python2.7/httplib.py:1053: in endheaders<br/>    self._send_output(message_body)<br/>/usr/lib/python2.7/httplib.py:897: in _send_output<br/>    self.send(msg)<br/>/usr/lib/python2.7/httplib.py:859: in send<br/>    self.connect()<br/>/usr/lib/python2.7/httplib.py:836: in connect<br/>    self.timeout, self.source_address)<br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/><br/>address = ('localhost', 11311), timeout = &lt;object object at 0x7f5b8b5b83b0&gt;, source_address = None<br/><br/>    def create_connection(address, timeout=_GLOBAL_DEFAULT_TIMEOUT,<br/>                          source_address=None):<br/>        """Connect to *address* and return the socket object.<br/>    <br/>        Convenience function.  Connect to *address* (a 2-tuple ``(host,<br/>        port)``) and return the socket object.  Passing the optional<br/>        *timeout* parameter will set the timeout on the socket instance<br/>        before attempting to connect.  If no *timeout* is supplied, the<br/>        global default timeout setting returned by :func:`getdefaulttimeout`<br/>        is used.  If *source_address* is set it must be a tuple of (host, port)<br/>        for the socket to bind as a source address before making the connection.<br/>        A host of '' or port 0 tells the OS to use the default.<br/>        """<br/>    <br/>        host, port = address<br/>        err = None<br/>        for res in getaddrinfo(host, port, 0, SOCK_STREAM):<br/>            af, socktype, proto, canonname, sa = res<br/>            sock = None<br/>            try:<br/>                sock = socket(af, socktype, proto)<br/>                if timeout is not _GLOBAL_DEFAULT_TIMEOUT:<br/>                    sock.settimeout(timeout)<br/>                if source_address:<br/>                    sock.bind(source_address)<br/>                sock.connect(sa)<br/>                return sock<br/>    <br/>            except error as _:<br/>                err = _<br/>                if sock is not None:<br/>                    sock.close()<br/>    <br/>        if err is not None:<br/>&gt;           raise err<br/><span class="error">E           error: [Errno 111] Connection refused</span><br/><br/>/usr/lib/python2.7/socket.py:575: error<br/></div></td></tr></tbody>
      <tbody class="error results-table-row">
        <tr>
          <td class="col-result">Error</td>
          <td class="col-name">tests/test_arm_utils.py::test_compute_ik[False]::setup</td>
          <td class="col-duration">0.00</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">@pytest.fixture(scope="module")<br/>    def create_robot():<br/>&gt;       return Robot('locobot', use_base=False)<br/><br/>test_arm_utils.py:15: <br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/core.py:105: in __init__<br/>    self.arm = arm_class(self.configs, **arm_config)<br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/locobot/arm.py:40: in __init__<br/>    use_arm = rospy.get_param('use_arm', False)<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/client.py:465: in get_param<br/>    return _param_server[param_name] #MasterProxy does all the magic for us<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/msproxy.py:121: in __getitem__<br/>    code, msg, value = self.target.getParam(rospy.names.get_caller_id(), resolved_key)<br/>/usr/lib/python2.7/xmlrpclib.py:1243: in __call__<br/>    return self.__send(self.__name, args)<br/>/usr/lib/python2.7/xmlrpclib.py:1602: in __request<br/>    verbose=self.__verbose<br/>/usr/lib/python2.7/xmlrpclib.py:1283: in request<br/>    return self.single_request(host, handler, request_body, verbose)<br/>/usr/lib/python2.7/xmlrpclib.py:1311: in single_request<br/>    self.send_content(h, request_body)<br/>/usr/lib/python2.7/xmlrpclib.py:1459: in send_content<br/>    connection.endheaders(request_body)<br/>/usr/lib/python2.7/httplib.py:1053: in endheaders<br/>    self._send_output(message_body)<br/>/usr/lib/python2.7/httplib.py:897: in _send_output<br/>    self.send(msg)<br/>/usr/lib/python2.7/httplib.py:859: in send<br/>    self.connect()<br/>/usr/lib/python2.7/httplib.py:836: in connect<br/>    self.timeout, self.source_address)<br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/><br/>address = ('localhost', 11311), timeout = &lt;object object at 0x7f5b8b5b83b0&gt;, source_address = None<br/><br/>    def create_connection(address, timeout=_GLOBAL_DEFAULT_TIMEOUT,<br/>                          source_address=None):<br/>        """Connect to *address* and return the socket object.<br/>    <br/>        Convenience function.  Connect to *address* (a 2-tuple ``(host,<br/>        port)``) and return the socket object.  Passing the optional<br/>        *timeout* parameter will set the timeout on the socket instance<br/>        before attempting to connect.  If no *timeout* is supplied, the<br/>        global default timeout setting returned by :func:`getdefaulttimeout`<br/>        is used.  If *source_address* is set it must be a tuple of (host, port)<br/>        for the socket to bind as a source address before making the connection.<br/>        A host of '' or port 0 tells the OS to use the default.<br/>        """<br/>    <br/>        host, port = address<br/>        err = None<br/>        for res in getaddrinfo(host, port, 0, SOCK_STREAM):<br/>            af, socktype, proto, canonname, sa = res<br/>            sock = None<br/>            try:<br/>                sock = socket(af, socktype, proto)<br/>                if timeout is not _GLOBAL_DEFAULT_TIMEOUT:<br/>                    sock.settimeout(timeout)<br/>                if source_address:<br/>                    sock.bind(source_address)<br/>                sock.connect(sa)<br/>                return sock<br/>    <br/>            except error as _:<br/>                err = _<br/>                if sock is not None:<br/>                    sock.close()<br/>    <br/>        if err is not None:<br/>&gt;           raise err<br/><span class="error">E           error: [Errno 111] Connection refused</span><br/><br/>/usr/lib/python2.7/socket.py:575: error<br/></div></td></tr></tbody>
      <tbody class="error results-table-row">
        <tr>
          <td class="col-result">Error</td>
          <td class="col-name">tests/test_arm_controls.py::test_position_control[True-position0]::setup</td>
          <td class="col-duration">0.00</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">@pytest.fixture(scope="module")<br/>    def create_robot():<br/>&gt;       return Robot('locobot', use_base=False)<br/><br/>test_arm_controls.py:17: <br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/core.py:105: in __init__<br/>    self.arm = arm_class(self.configs, **arm_config)<br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/locobot/arm.py:40: in __init__<br/>    use_arm = rospy.get_param('use_arm', False)<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/client.py:465: in get_param<br/>    return _param_server[param_name] #MasterProxy does all the magic for us<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/msproxy.py:121: in __getitem__<br/>    code, msg, value = self.target.getParam(rospy.names.get_caller_id(), resolved_key)<br/>/usr/lib/python2.7/xmlrpclib.py:1243: in __call__<br/>    return self.__send(self.__name, args)<br/>/usr/lib/python2.7/xmlrpclib.py:1602: in __request<br/>    verbose=self.__verbose<br/>/usr/lib/python2.7/xmlrpclib.py:1283: in request<br/>    return self.single_request(host, handler, request_body, verbose)<br/>/usr/lib/python2.7/xmlrpclib.py:1311: in single_request<br/>    self.send_content(h, request_body)<br/>/usr/lib/python2.7/xmlrpclib.py:1459: in send_content<br/>    connection.endheaders(request_body)<br/>/usr/lib/python2.7/httplib.py:1053: in endheaders<br/>    self._send_output(message_body)<br/>/usr/lib/python2.7/httplib.py:897: in _send_output<br/>    self.send(msg)<br/>/usr/lib/python2.7/httplib.py:859: in send<br/>    self.connect()<br/>/usr/lib/python2.7/httplib.py:836: in connect<br/>    self.timeout, self.source_address)<br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/><br/>address = ('localhost', 11311), timeout = &lt;object object at 0x7f5b8b5b83b0&gt;, source_address = None<br/><br/>    def create_connection(address, timeout=_GLOBAL_DEFAULT_TIMEOUT,<br/>                          source_address=None):<br/>        """Connect to *address* and return the socket object.<br/>    <br/>        Convenience function.  Connect to *address* (a 2-tuple ``(host,<br/>        port)``) and return the socket object.  Passing the optional<br/>        *timeout* parameter will set the timeout on the socket instance<br/>        before attempting to connect.  If no *timeout* is supplied, the<br/>        global default timeout setting returned by :func:`getdefaulttimeout`<br/>        is used.  If *source_address* is set it must be a tuple of (host, port)<br/>        for the socket to bind as a source address before making the connection.<br/>        A host of '' or port 0 tells the OS to use the default.<br/>        """<br/>    <br/>        host, port = address<br/>        err = None<br/>        for res in getaddrinfo(host, port, 0, SOCK_STREAM):<br/>            af, socktype, proto, canonname, sa = res<br/>            sock = None<br/>            try:<br/>                sock = socket(af, socktype, proto)<br/>                if timeout is not _GLOBAL_DEFAULT_TIMEOUT:<br/>                    sock.settimeout(timeout)<br/>                if source_address:<br/>                    sock.bind(source_address)<br/>                sock.connect(sa)<br/>                return sock<br/>    <br/>            except error as _:<br/>                err = _<br/>                if sock is not None:<br/>                    sock.close()<br/>    <br/>        if err is not None:<br/>&gt;           raise err<br/><span class="error">E           error: [Errno 111] Connection refused</span><br/><br/>/usr/lib/python2.7/socket.py:575: error<br/>---------------------------- Captured stdout setup -----------------------------<br/>['husky', 'locobot', 'sawyer', 'turtlebot']
</div></td></tr></tbody>
      <tbody class="error results-table-row">
        <tr>
          <td class="col-result">Error</td>
          <td class="col-name">tests/test_arm_controls.py::test_position_control[True-position1]::setup</td>
          <td class="col-duration">0.00</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">@pytest.fixture(scope="module")<br/>    def create_robot():<br/>&gt;       return Robot('locobot', use_base=False)<br/><br/>test_arm_controls.py:17: <br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/core.py:105: in __init__<br/>    self.arm = arm_class(self.configs, **arm_config)<br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/locobot/arm.py:40: in __init__<br/>    use_arm = rospy.get_param('use_arm', False)<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/client.py:465: in get_param<br/>    return _param_server[param_name] #MasterProxy does all the magic for us<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/msproxy.py:121: in __getitem__<br/>    code, msg, value = self.target.getParam(rospy.names.get_caller_id(), resolved_key)<br/>/usr/lib/python2.7/xmlrpclib.py:1243: in __call__<br/>    return self.__send(self.__name, args)<br/>/usr/lib/python2.7/xmlrpclib.py:1602: in __request<br/>    verbose=self.__verbose<br/>/usr/lib/python2.7/xmlrpclib.py:1283: in request<br/>    return self.single_request(host, handler, request_body, verbose)<br/>/usr/lib/python2.7/xmlrpclib.py:1311: in single_request<br/>    self.send_content(h, request_body)<br/>/usr/lib/python2.7/xmlrpclib.py:1459: in send_content<br/>    connection.endheaders(request_body)<br/>/usr/lib/python2.7/httplib.py:1053: in endheaders<br/>    self._send_output(message_body)<br/>/usr/lib/python2.7/httplib.py:897: in _send_output<br/>    self.send(msg)<br/>/usr/lib/python2.7/httplib.py:859: in send<br/>    self.connect()<br/>/usr/lib/python2.7/httplib.py:836: in connect<br/>    self.timeout, self.source_address)<br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/><br/>address = ('localhost', 11311), timeout = &lt;object object at 0x7f5b8b5b83b0&gt;, source_address = None<br/><br/>    def create_connection(address, timeout=_GLOBAL_DEFAULT_TIMEOUT,<br/>                          source_address=None):<br/>        """Connect to *address* and return the socket object.<br/>    <br/>        Convenience function.  Connect to *address* (a 2-tuple ``(host,<br/>        port)``) and return the socket object.  Passing the optional<br/>        *timeout* parameter will set the timeout on the socket instance<br/>        before attempting to connect.  If no *timeout* is supplied, the<br/>        global default timeout setting returned by :func:`getdefaulttimeout`<br/>        is used.  If *source_address* is set it must be a tuple of (host, port)<br/>        for the socket to bind as a source address before making the connection.<br/>        A host of '' or port 0 tells the OS to use the default.<br/>        """<br/>    <br/>        host, port = address<br/>        err = None<br/>        for res in getaddrinfo(host, port, 0, SOCK_STREAM):<br/>            af, socktype, proto, canonname, sa = res<br/>            sock = None<br/>            try:<br/>                sock = socket(af, socktype, proto)<br/>                if timeout is not _GLOBAL_DEFAULT_TIMEOUT:<br/>                    sock.settimeout(timeout)<br/>                if source_address:<br/>                    sock.bind(source_address)<br/>                sock.connect(sa)<br/>                return sock<br/>    <br/>            except error as _:<br/>                err = _<br/>                if sock is not None:<br/>                    sock.close()<br/>    <br/>        if err is not None:<br/>&gt;           raise err<br/><span class="error">E           error: [Errno 111] Connection refused</span><br/><br/>/usr/lib/python2.7/socket.py:575: error<br/></div></td></tr></tbody>
      <tbody class="error results-table-row">
        <tr>
          <td class="col-result">Error</td>
          <td class="col-name">tests/test_arm_controls.py::test_position_control[False-position0]::setup</td>
          <td class="col-duration">0.00</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">@pytest.fixture(scope="module")<br/>    def create_robot():<br/>&gt;       return Robot('locobot', use_base=False)<br/><br/>test_arm_controls.py:17: <br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/core.py:105: in __init__<br/>    self.arm = arm_class(self.configs, **arm_config)<br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/locobot/arm.py:40: in __init__<br/>    use_arm = rospy.get_param('use_arm', False)<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/client.py:465: in get_param<br/>    return _param_server[param_name] #MasterProxy does all the magic for us<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/msproxy.py:121: in __getitem__<br/>    code, msg, value = self.target.getParam(rospy.names.get_caller_id(), resolved_key)<br/>/usr/lib/python2.7/xmlrpclib.py:1243: in __call__<br/>    return self.__send(self.__name, args)<br/>/usr/lib/python2.7/xmlrpclib.py:1602: in __request<br/>    verbose=self.__verbose<br/>/usr/lib/python2.7/xmlrpclib.py:1283: in request<br/>    return self.single_request(host, handler, request_body, verbose)<br/>/usr/lib/python2.7/xmlrpclib.py:1311: in single_request<br/>    self.send_content(h, request_body)<br/>/usr/lib/python2.7/xmlrpclib.py:1459: in send_content<br/>    connection.endheaders(request_body)<br/>/usr/lib/python2.7/httplib.py:1053: in endheaders<br/>    self._send_output(message_body)<br/>/usr/lib/python2.7/httplib.py:897: in _send_output<br/>    self.send(msg)<br/>/usr/lib/python2.7/httplib.py:859: in send<br/>    self.connect()<br/>/usr/lib/python2.7/httplib.py:836: in connect<br/>    self.timeout, self.source_address)<br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/><br/>address = ('localhost', 11311), timeout = &lt;object object at 0x7f5b8b5b83b0&gt;, source_address = None<br/><br/>    def create_connection(address, timeout=_GLOBAL_DEFAULT_TIMEOUT,<br/>                          source_address=None):<br/>        """Connect to *address* and return the socket object.<br/>    <br/>        Convenience function.  Connect to *address* (a 2-tuple ``(host,<br/>        port)``) and return the socket object.  Passing the optional<br/>        *timeout* parameter will set the timeout on the socket instance<br/>        before attempting to connect.  If no *timeout* is supplied, the<br/>        global default timeout setting returned by :func:`getdefaulttimeout`<br/>        is used.  If *source_address* is set it must be a tuple of (host, port)<br/>        for the socket to bind as a source address before making the connection.<br/>        A host of '' or port 0 tells the OS to use the default.<br/>        """<br/>    <br/>        host, port = address<br/>        err = None<br/>        for res in getaddrinfo(host, port, 0, SOCK_STREAM):<br/>            af, socktype, proto, canonname, sa = res<br/>            sock = None<br/>            try:<br/>                sock = socket(af, socktype, proto)<br/>                if timeout is not _GLOBAL_DEFAULT_TIMEOUT:<br/>                    sock.settimeout(timeout)<br/>                if source_address:<br/>                    sock.bind(source_address)<br/>                sock.connect(sa)<br/>                return sock<br/>    <br/>            except error as _:<br/>                err = _<br/>                if sock is not None:<br/>                    sock.close()<br/>    <br/>        if err is not None:<br/>&gt;           raise err<br/><span class="error">E           error: [Errno 111] Connection refused</span><br/><br/>/usr/lib/python2.7/socket.py:575: error<br/></div></td></tr></tbody>
      <tbody class="error results-table-row">
        <tr>
          <td class="col-result">Error</td>
          <td class="col-name">tests/test_arm_controls.py::test_position_control[False-position1]::setup</td>
          <td class="col-duration">0.00</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">@pytest.fixture(scope="module")<br/>    def create_robot():<br/>&gt;       return Robot('locobot', use_base=False)<br/><br/>test_arm_controls.py:17: <br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/core.py:105: in __init__<br/>    self.arm = arm_class(self.configs, **arm_config)<br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/locobot/arm.py:40: in __init__<br/>    use_arm = rospy.get_param('use_arm', False)<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/client.py:465: in get_param<br/>    return _param_server[param_name] #MasterProxy does all the magic for us<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/msproxy.py:121: in __getitem__<br/>    code, msg, value = self.target.getParam(rospy.names.get_caller_id(), resolved_key)<br/>/usr/lib/python2.7/xmlrpclib.py:1243: in __call__<br/>    return self.__send(self.__name, args)<br/>/usr/lib/python2.7/xmlrpclib.py:1602: in __request<br/>    verbose=self.__verbose<br/>/usr/lib/python2.7/xmlrpclib.py:1283: in request<br/>    return self.single_request(host, handler, request_body, verbose)<br/>/usr/lib/python2.7/xmlrpclib.py:1311: in single_request<br/>    self.send_content(h, request_body)<br/>/usr/lib/python2.7/xmlrpclib.py:1459: in send_content<br/>    connection.endheaders(request_body)<br/>/usr/lib/python2.7/httplib.py:1053: in endheaders<br/>    self._send_output(message_body)<br/>/usr/lib/python2.7/httplib.py:897: in _send_output<br/>    self.send(msg)<br/>/usr/lib/python2.7/httplib.py:859: in send<br/>    self.connect()<br/>/usr/lib/python2.7/httplib.py:836: in connect<br/>    self.timeout, self.source_address)<br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/><br/>address = ('localhost', 11311), timeout = &lt;object object at 0x7f5b8b5b83b0&gt;, source_address = None<br/><br/>    def create_connection(address, timeout=_GLOBAL_DEFAULT_TIMEOUT,<br/>                          source_address=None):<br/>        """Connect to *address* and return the socket object.<br/>    <br/>        Convenience function.  Connect to *address* (a 2-tuple ``(host,<br/>        port)``) and return the socket object.  Passing the optional<br/>        *timeout* parameter will set the timeout on the socket instance<br/>        before attempting to connect.  If no *timeout* is supplied, the<br/>        global default timeout setting returned by :func:`getdefaulttimeout`<br/>        is used.  If *source_address* is set it must be a tuple of (host, port)<br/>        for the socket to bind as a source address before making the connection.<br/>        A host of '' or port 0 tells the OS to use the default.<br/>        """<br/>    <br/>        host, port = address<br/>        err = None<br/>        for res in getaddrinfo(host, port, 0, SOCK_STREAM):<br/>            af, socktype, proto, canonname, sa = res<br/>            sock = None<br/>            try:<br/>                sock = socket(af, socktype, proto)<br/>                if timeout is not _GLOBAL_DEFAULT_TIMEOUT:<br/>                    sock.settimeout(timeout)<br/>                if source_address:<br/>                    sock.bind(source_address)<br/>                sock.connect(sa)<br/>                return sock<br/>    <br/>            except error as _:<br/>                err = _<br/>                if sock is not None:<br/>                    sock.close()<br/>    <br/>        if err is not None:<br/>&gt;           raise err<br/><span class="error">E           error: [Errno 111] Connection refused</span><br/><br/>/usr/lib/python2.7/socket.py:575: error<br/></div></td></tr></tbody>
      <tbody class="error results-table-row">
        <tr>
          <td class="col-result">Error</td>
          <td class="col-name">tests/test_arm_controls.py::test_ee_pose_control[True-position0-orientation0]::setup</td>
          <td class="col-duration">0.00</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">@pytest.fixture(scope="module")<br/>    def create_robot():<br/>&gt;       return Robot('locobot', use_base=False)<br/><br/>test_arm_controls.py:17: <br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/core.py:105: in __init__<br/>    self.arm = arm_class(self.configs, **arm_config)<br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/locobot/arm.py:40: in __init__<br/>    use_arm = rospy.get_param('use_arm', False)<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/client.py:465: in get_param<br/>    return _param_server[param_name] #MasterProxy does all the magic for us<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/msproxy.py:121: in __getitem__<br/>    code, msg, value = self.target.getParam(rospy.names.get_caller_id(), resolved_key)<br/>/usr/lib/python2.7/xmlrpclib.py:1243: in __call__<br/>    return self.__send(self.__name, args)<br/>/usr/lib/python2.7/xmlrpclib.py:1602: in __request<br/>    verbose=self.__verbose<br/>/usr/lib/python2.7/xmlrpclib.py:1283: in request<br/>    return self.single_request(host, handler, request_body, verbose)<br/>/usr/lib/python2.7/xmlrpclib.py:1311: in single_request<br/>    self.send_content(h, request_body)<br/>/usr/lib/python2.7/xmlrpclib.py:1459: in send_content<br/>    connection.endheaders(request_body)<br/>/usr/lib/python2.7/httplib.py:1053: in endheaders<br/>    self._send_output(message_body)<br/>/usr/lib/python2.7/httplib.py:897: in _send_output<br/>    self.send(msg)<br/>/usr/lib/python2.7/httplib.py:859: in send<br/>    self.connect()<br/>/usr/lib/python2.7/httplib.py:836: in connect<br/>    self.timeout, self.source_address)<br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/><br/>address = ('localhost', 11311), timeout = &lt;object object at 0x7f5b8b5b83b0&gt;, source_address = None<br/><br/>    def create_connection(address, timeout=_GLOBAL_DEFAULT_TIMEOUT,<br/>                          source_address=None):<br/>        """Connect to *address* and return the socket object.<br/>    <br/>        Convenience function.  Connect to *address* (a 2-tuple ``(host,<br/>        port)``) and return the socket object.  Passing the optional<br/>        *timeout* parameter will set the timeout on the socket instance<br/>        before attempting to connect.  If no *timeout* is supplied, the<br/>        global default timeout setting returned by :func:`getdefaulttimeout`<br/>        is used.  If *source_address* is set it must be a tuple of (host, port)<br/>        for the socket to bind as a source address before making the connection.<br/>        A host of '' or port 0 tells the OS to use the default.<br/>        """<br/>    <br/>        host, port = address<br/>        err = None<br/>        for res in getaddrinfo(host, port, 0, SOCK_STREAM):<br/>            af, socktype, proto, canonname, sa = res<br/>            sock = None<br/>            try:<br/>                sock = socket(af, socktype, proto)<br/>                if timeout is not _GLOBAL_DEFAULT_TIMEOUT:<br/>                    sock.settimeout(timeout)<br/>                if source_address:<br/>                    sock.bind(source_address)<br/>                sock.connect(sa)<br/>                return sock<br/>    <br/>            except error as _:<br/>                err = _<br/>                if sock is not None:<br/>                    sock.close()<br/>    <br/>        if err is not None:<br/>&gt;           raise err<br/><span class="error">E           error: [Errno 111] Connection refused</span><br/><br/>/usr/lib/python2.7/socket.py:575: error<br/></div></td></tr></tbody>
      <tbody class="error results-table-row">
        <tr>
          <td class="col-result">Error</td>
          <td class="col-name">tests/test_arm_controls.py::test_ee_pose_control[True-position1-orientation1]::setup</td>
          <td class="col-duration">0.00</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">@pytest.fixture(scope="module")<br/>    def create_robot():<br/>&gt;       return Robot('locobot', use_base=False)<br/><br/>test_arm_controls.py:17: <br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/core.py:105: in __init__<br/>    self.arm = arm_class(self.configs, **arm_config)<br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/locobot/arm.py:40: in __init__<br/>    use_arm = rospy.get_param('use_arm', False)<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/client.py:465: in get_param<br/>    return _param_server[param_name] #MasterProxy does all the magic for us<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/msproxy.py:121: in __getitem__<br/>    code, msg, value = self.target.getParam(rospy.names.get_caller_id(), resolved_key)<br/>/usr/lib/python2.7/xmlrpclib.py:1243: in __call__<br/>    return self.__send(self.__name, args)<br/>/usr/lib/python2.7/xmlrpclib.py:1602: in __request<br/>    verbose=self.__verbose<br/>/usr/lib/python2.7/xmlrpclib.py:1283: in request<br/>    return self.single_request(host, handler, request_body, verbose)<br/>/usr/lib/python2.7/xmlrpclib.py:1311: in single_request<br/>    self.send_content(h, request_body)<br/>/usr/lib/python2.7/xmlrpclib.py:1459: in send_content<br/>    connection.endheaders(request_body)<br/>/usr/lib/python2.7/httplib.py:1053: in endheaders<br/>    self._send_output(message_body)<br/>/usr/lib/python2.7/httplib.py:897: in _send_output<br/>    self.send(msg)<br/>/usr/lib/python2.7/httplib.py:859: in send<br/>    self.connect()<br/>/usr/lib/python2.7/httplib.py:836: in connect<br/>    self.timeout, self.source_address)<br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/><br/>address = ('localhost', 11311), timeout = &lt;object object at 0x7f5b8b5b83b0&gt;, source_address = None<br/><br/>    def create_connection(address, timeout=_GLOBAL_DEFAULT_TIMEOUT,<br/>                          source_address=None):<br/>        """Connect to *address* and return the socket object.<br/>    <br/>        Convenience function.  Connect to *address* (a 2-tuple ``(host,<br/>        port)``) and return the socket object.  Passing the optional<br/>        *timeout* parameter will set the timeout on the socket instance<br/>        before attempting to connect.  If no *timeout* is supplied, the<br/>        global default timeout setting returned by :func:`getdefaulttimeout`<br/>        is used.  If *source_address* is set it must be a tuple of (host, port)<br/>        for the socket to bind as a source address before making the connection.<br/>        A host of '' or port 0 tells the OS to use the default.<br/>        """<br/>    <br/>        host, port = address<br/>        err = None<br/>        for res in getaddrinfo(host, port, 0, SOCK_STREAM):<br/>            af, socktype, proto, canonname, sa = res<br/>            sock = None<br/>            try:<br/>                sock = socket(af, socktype, proto)<br/>                if timeout is not _GLOBAL_DEFAULT_TIMEOUT:<br/>                    sock.settimeout(timeout)<br/>                if source_address:<br/>                    sock.bind(source_address)<br/>                sock.connect(sa)<br/>                return sock<br/>    <br/>            except error as _:<br/>                err = _<br/>                if sock is not None:<br/>                    sock.close()<br/>    <br/>        if err is not None:<br/>&gt;           raise err<br/><span class="error">E           error: [Errno 111] Connection refused</span><br/><br/>/usr/lib/python2.7/socket.py:575: error<br/></div></td></tr></tbody>
      <tbody class="error results-table-row">
        <tr>
          <td class="col-result">Error</td>
          <td class="col-name">tests/test_arm_controls.py::test_ee_pose_control[False-position0-orientation0]::setup</td>
          <td class="col-duration">0.00</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">@pytest.fixture(scope="module")<br/>    def create_robot():<br/>&gt;       return Robot('locobot', use_base=False)<br/><br/>test_arm_controls.py:17: <br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/core.py:105: in __init__<br/>    self.arm = arm_class(self.configs, **arm_config)<br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/locobot/arm.py:40: in __init__<br/>    use_arm = rospy.get_param('use_arm', False)<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/client.py:465: in get_param<br/>    return _param_server[param_name] #MasterProxy does all the magic for us<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/msproxy.py:121: in __getitem__<br/>    code, msg, value = self.target.getParam(rospy.names.get_caller_id(), resolved_key)<br/>/usr/lib/python2.7/xmlrpclib.py:1243: in __call__<br/>    return self.__send(self.__name, args)<br/>/usr/lib/python2.7/xmlrpclib.py:1602: in __request<br/>    verbose=self.__verbose<br/>/usr/lib/python2.7/xmlrpclib.py:1283: in request<br/>    return self.single_request(host, handler, request_body, verbose)<br/>/usr/lib/python2.7/xmlrpclib.py:1311: in single_request<br/>    self.send_content(h, request_body)<br/>/usr/lib/python2.7/xmlrpclib.py:1459: in send_content<br/>    connection.endheaders(request_body)<br/>/usr/lib/python2.7/httplib.py:1053: in endheaders<br/>    self._send_output(message_body)<br/>/usr/lib/python2.7/httplib.py:897: in _send_output<br/>    self.send(msg)<br/>/usr/lib/python2.7/httplib.py:859: in send<br/>    self.connect()<br/>/usr/lib/python2.7/httplib.py:836: in connect<br/>    self.timeout, self.source_address)<br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/><br/>address = ('localhost', 11311), timeout = &lt;object object at 0x7f5b8b5b83b0&gt;, source_address = None<br/><br/>    def create_connection(address, timeout=_GLOBAL_DEFAULT_TIMEOUT,<br/>                          source_address=None):<br/>        """Connect to *address* and return the socket object.<br/>    <br/>        Convenience function.  Connect to *address* (a 2-tuple ``(host,<br/>        port)``) and return the socket object.  Passing the optional<br/>        *timeout* parameter will set the timeout on the socket instance<br/>        before attempting to connect.  If no *timeout* is supplied, the<br/>        global default timeout setting returned by :func:`getdefaulttimeout`<br/>        is used.  If *source_address* is set it must be a tuple of (host, port)<br/>        for the socket to bind as a source address before making the connection.<br/>        A host of '' or port 0 tells the OS to use the default.<br/>        """<br/>    <br/>        host, port = address<br/>        err = None<br/>        for res in getaddrinfo(host, port, 0, SOCK_STREAM):<br/>            af, socktype, proto, canonname, sa = res<br/>            sock = None<br/>            try:<br/>                sock = socket(af, socktype, proto)<br/>                if timeout is not _GLOBAL_DEFAULT_TIMEOUT:<br/>                    sock.settimeout(timeout)<br/>                if source_address:<br/>                    sock.bind(source_address)<br/>                sock.connect(sa)<br/>                return sock<br/>    <br/>            except error as _:<br/>                err = _<br/>                if sock is not None:<br/>                    sock.close()<br/>    <br/>        if err is not None:<br/>&gt;           raise err<br/><span class="error">E           error: [Errno 111] Connection refused</span><br/><br/>/usr/lib/python2.7/socket.py:575: error<br/></div></td></tr></tbody>
      <tbody class="error results-table-row">
        <tr>
          <td class="col-result">Error</td>
          <td class="col-name">tests/test_arm_controls.py::test_ee_pose_control[False-position1-orientation1]::setup</td>
          <td class="col-duration">0.00</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">@pytest.fixture(scope="module")<br/>    def create_robot():<br/>&gt;       return Robot('locobot', use_base=False)<br/><br/>test_arm_controls.py:17: <br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/core.py:105: in __init__<br/>    self.arm = arm_class(self.configs, **arm_config)<br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/locobot/arm.py:40: in __init__<br/>    use_arm = rospy.get_param('use_arm', False)<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/client.py:465: in get_param<br/>    return _param_server[param_name] #MasterProxy does all the magic for us<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/msproxy.py:121: in __getitem__<br/>    code, msg, value = self.target.getParam(rospy.names.get_caller_id(), resolved_key)<br/>/usr/lib/python2.7/xmlrpclib.py:1243: in __call__<br/>    return self.__send(self.__name, args)<br/>/usr/lib/python2.7/xmlrpclib.py:1602: in __request<br/>    verbose=self.__verbose<br/>/usr/lib/python2.7/xmlrpclib.py:1283: in request<br/>    return self.single_request(host, handler, request_body, verbose)<br/>/usr/lib/python2.7/xmlrpclib.py:1311: in single_request<br/>    self.send_content(h, request_body)<br/>/usr/lib/python2.7/xmlrpclib.py:1459: in send_content<br/>    connection.endheaders(request_body)<br/>/usr/lib/python2.7/httplib.py:1053: in endheaders<br/>    self._send_output(message_body)<br/>/usr/lib/python2.7/httplib.py:897: in _send_output<br/>    self.send(msg)<br/>/usr/lib/python2.7/httplib.py:859: in send<br/>    self.connect()<br/>/usr/lib/python2.7/httplib.py:836: in connect<br/>    self.timeout, self.source_address)<br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/><br/>address = ('localhost', 11311), timeout = &lt;object object at 0x7f5b8b5b83b0&gt;, source_address = None<br/><br/>    def create_connection(address, timeout=_GLOBAL_DEFAULT_TIMEOUT,<br/>                          source_address=None):<br/>        """Connect to *address* and return the socket object.<br/>    <br/>        Convenience function.  Connect to *address* (a 2-tuple ``(host,<br/>        port)``) and return the socket object.  Passing the optional<br/>        *timeout* parameter will set the timeout on the socket instance<br/>        before attempting to connect.  If no *timeout* is supplied, the<br/>        global default timeout setting returned by :func:`getdefaulttimeout`<br/>        is used.  If *source_address* is set it must be a tuple of (host, port)<br/>        for the socket to bind as a source address before making the connection.<br/>        A host of '' or port 0 tells the OS to use the default.<br/>        """<br/>    <br/>        host, port = address<br/>        err = None<br/>        for res in getaddrinfo(host, port, 0, SOCK_STREAM):<br/>            af, socktype, proto, canonname, sa = res<br/>            sock = None<br/>            try:<br/>                sock = socket(af, socktype, proto)<br/>                if timeout is not _GLOBAL_DEFAULT_TIMEOUT:<br/>                    sock.settimeout(timeout)<br/>                if source_address:<br/>                    sock.bind(source_address)<br/>                sock.connect(sa)<br/>                return sock<br/>    <br/>            except error as _:<br/>                err = _<br/>                if sock is not None:<br/>                    sock.close()<br/>    <br/>        if err is not None:<br/>&gt;           raise err<br/><span class="error">E           error: [Errno 111] Connection refused</span><br/><br/>/usr/lib/python2.7/socket.py:575: error<br/></div></td></tr></tbody>
      <tbody class="error results-table-row">
        <tr>
          <td class="col-result">Error</td>
          <td class="col-name">tests/test_arm_controls.py::test_ee_pitch_control[position0-0.5-None-tgt_pos0-tgt_quat0]::setup</td>
          <td class="col-duration">0.00</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">@pytest.fixture(scope="module")<br/>    def create_robot():<br/>&gt;       return Robot('locobot', use_base=False)<br/><br/>test_arm_controls.py:17: <br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/core.py:105: in __init__<br/>    self.arm = arm_class(self.configs, **arm_config)<br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/locobot/arm.py:40: in __init__<br/>    use_arm = rospy.get_param('use_arm', False)<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/client.py:465: in get_param<br/>    return _param_server[param_name] #MasterProxy does all the magic for us<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/msproxy.py:121: in __getitem__<br/>    code, msg, value = self.target.getParam(rospy.names.get_caller_id(), resolved_key)<br/>/usr/lib/python2.7/xmlrpclib.py:1243: in __call__<br/>    return self.__send(self.__name, args)<br/>/usr/lib/python2.7/xmlrpclib.py:1602: in __request<br/>    verbose=self.__verbose<br/>/usr/lib/python2.7/xmlrpclib.py:1283: in request<br/>    return self.single_request(host, handler, request_body, verbose)<br/>/usr/lib/python2.7/xmlrpclib.py:1311: in single_request<br/>    self.send_content(h, request_body)<br/>/usr/lib/python2.7/xmlrpclib.py:1459: in send_content<br/>    connection.endheaders(request_body)<br/>/usr/lib/python2.7/httplib.py:1053: in endheaders<br/>    self._send_output(message_body)<br/>/usr/lib/python2.7/httplib.py:897: in _send_output<br/>    self.send(msg)<br/>/usr/lib/python2.7/httplib.py:859: in send<br/>    self.connect()<br/>/usr/lib/python2.7/httplib.py:836: in connect<br/>    self.timeout, self.source_address)<br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/><br/>address = ('localhost', 11311), timeout = &lt;object object at 0x7f5b8b5b83b0&gt;, source_address = None<br/><br/>    def create_connection(address, timeout=_GLOBAL_DEFAULT_TIMEOUT,<br/>                          source_address=None):<br/>        """Connect to *address* and return the socket object.<br/>    <br/>        Convenience function.  Connect to *address* (a 2-tuple ``(host,<br/>        port)``) and return the socket object.  Passing the optional<br/>        *timeout* parameter will set the timeout on the socket instance<br/>        before attempting to connect.  If no *timeout* is supplied, the<br/>        global default timeout setting returned by :func:`getdefaulttimeout`<br/>        is used.  If *source_address* is set it must be a tuple of (host, port)<br/>        for the socket to bind as a source address before making the connection.<br/>        A host of '' or port 0 tells the OS to use the default.<br/>        """<br/>    <br/>        host, port = address<br/>        err = None<br/>        for res in getaddrinfo(host, port, 0, SOCK_STREAM):<br/>            af, socktype, proto, canonname, sa = res<br/>            sock = None<br/>            try:<br/>                sock = socket(af, socktype, proto)<br/>                if timeout is not _GLOBAL_DEFAULT_TIMEOUT:<br/>                    sock.settimeout(timeout)<br/>                if source_address:<br/>                    sock.bind(source_address)<br/>                sock.connect(sa)<br/>                return sock<br/>    <br/>            except error as _:<br/>                err = _<br/>                if sock is not None:<br/>                    sock.close()<br/>    <br/>        if err is not None:<br/>&gt;           raise err<br/><span class="error">E           error: [Errno 111] Connection refused</span><br/><br/>/usr/lib/python2.7/socket.py:575: error<br/></div></td></tr></tbody>
      <tbody class="error results-table-row">
        <tr>
          <td class="col-result">Error</td>
          <td class="col-name">tests/test_arm_controls.py::test_ee_pitch_control[position1-0.5-0.5-tgt_pos1-tgt_quat1]::setup</td>
          <td class="col-duration">0.00</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">@pytest.fixture(scope="module")<br/>    def create_robot():<br/>&gt;       return Robot('locobot', use_base=False)<br/><br/>test_arm_controls.py:17: <br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/core.py:105: in __init__<br/>    self.arm = arm_class(self.configs, **arm_config)<br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/locobot/arm.py:40: in __init__<br/>    use_arm = rospy.get_param('use_arm', False)<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/client.py:465: in get_param<br/>    return _param_server[param_name] #MasterProxy does all the magic for us<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/msproxy.py:121: in __getitem__<br/>    code, msg, value = self.target.getParam(rospy.names.get_caller_id(), resolved_key)<br/>/usr/lib/python2.7/xmlrpclib.py:1243: in __call__<br/>    return self.__send(self.__name, args)<br/>/usr/lib/python2.7/xmlrpclib.py:1602: in __request<br/>    verbose=self.__verbose<br/>/usr/lib/python2.7/xmlrpclib.py:1283: in request<br/>    return self.single_request(host, handler, request_body, verbose)<br/>/usr/lib/python2.7/xmlrpclib.py:1311: in single_request<br/>    self.send_content(h, request_body)<br/>/usr/lib/python2.7/xmlrpclib.py:1459: in send_content<br/>    connection.endheaders(request_body)<br/>/usr/lib/python2.7/httplib.py:1053: in endheaders<br/>    self._send_output(message_body)<br/>/usr/lib/python2.7/httplib.py:897: in _send_output<br/>    self.send(msg)<br/>/usr/lib/python2.7/httplib.py:859: in send<br/>    self.connect()<br/>/usr/lib/python2.7/httplib.py:836: in connect<br/>    self.timeout, self.source_address)<br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/><br/>address = ('localhost', 11311), timeout = &lt;object object at 0x7f5b8b5b83b0&gt;, source_address = None<br/><br/>    def create_connection(address, timeout=_GLOBAL_DEFAULT_TIMEOUT,<br/>                          source_address=None):<br/>        """Connect to *address* and return the socket object.<br/>    <br/>        Convenience function.  Connect to *address* (a 2-tuple ``(host,<br/>        port)``) and return the socket object.  Passing the optional<br/>        *timeout* parameter will set the timeout on the socket instance<br/>        before attempting to connect.  If no *timeout* is supplied, the<br/>        global default timeout setting returned by :func:`getdefaulttimeout`<br/>        is used.  If *source_address* is set it must be a tuple of (host, port)<br/>        for the socket to bind as a source address before making the connection.<br/>        A host of '' or port 0 tells the OS to use the default.<br/>        """<br/>    <br/>        host, port = address<br/>        err = None<br/>        for res in getaddrinfo(host, port, 0, SOCK_STREAM):<br/>            af, socktype, proto, canonname, sa = res<br/>            sock = None<br/>            try:<br/>                sock = socket(af, socktype, proto)<br/>                if timeout is not _GLOBAL_DEFAULT_TIMEOUT:<br/>                    sock.settimeout(timeout)<br/>                if source_address:<br/>                    sock.bind(source_address)<br/>                sock.connect(sa)<br/>                return sock<br/>    <br/>            except error as _:<br/>                err = _<br/>                if sock is not None:<br/>                    sock.close()<br/>    <br/>        if err is not None:<br/>&gt;           raise err<br/><span class="error">E           error: [Errno 111] Connection refused</span><br/><br/>/usr/lib/python2.7/socket.py:575: error<br/></div></td></tr></tbody>
      <tbody class="error results-table-row">
        <tr>
          <td class="col-result">Error</td>
          <td class="col-name">tests/test_arm_controls.py::test_ee_pitch_control[position2-1.57079632679-0.785398163397-tgt_pos2-tgt_quat2]::setup</td>
          <td class="col-duration">0.00</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">@pytest.fixture(scope="module")<br/>    def create_robot():<br/>&gt;       return Robot('locobot', use_base=False)<br/><br/>test_arm_controls.py:17: <br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/core.py:105: in __init__<br/>    self.arm = arm_class(self.configs, **arm_config)<br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/locobot/arm.py:40: in __init__<br/>    use_arm = rospy.get_param('use_arm', False)<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/client.py:465: in get_param<br/>    return _param_server[param_name] #MasterProxy does all the magic for us<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/msproxy.py:121: in __getitem__<br/>    code, msg, value = self.target.getParam(rospy.names.get_caller_id(), resolved_key)<br/>/usr/lib/python2.7/xmlrpclib.py:1243: in __call__<br/>    return self.__send(self.__name, args)<br/>/usr/lib/python2.7/xmlrpclib.py:1602: in __request<br/>    verbose=self.__verbose<br/>/usr/lib/python2.7/xmlrpclib.py:1283: in request<br/>    return self.single_request(host, handler, request_body, verbose)<br/>/usr/lib/python2.7/xmlrpclib.py:1311: in single_request<br/>    self.send_content(h, request_body)<br/>/usr/lib/python2.7/xmlrpclib.py:1459: in send_content<br/>    connection.endheaders(request_body)<br/>/usr/lib/python2.7/httplib.py:1053: in endheaders<br/>    self._send_output(message_body)<br/>/usr/lib/python2.7/httplib.py:897: in _send_output<br/>    self.send(msg)<br/>/usr/lib/python2.7/httplib.py:859: in send<br/>    self.connect()<br/>/usr/lib/python2.7/httplib.py:836: in connect<br/>    self.timeout, self.source_address)<br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/><br/>address = ('localhost', 11311), timeout = &lt;object object at 0x7f5b8b5b83b0&gt;, source_address = None<br/><br/>    def create_connection(address, timeout=_GLOBAL_DEFAULT_TIMEOUT,<br/>                          source_address=None):<br/>        """Connect to *address* and return the socket object.<br/>    <br/>        Convenience function.  Connect to *address* (a 2-tuple ``(host,<br/>        port)``) and return the socket object.  Passing the optional<br/>        *timeout* parameter will set the timeout on the socket instance<br/>        before attempting to connect.  If no *timeout* is supplied, the<br/>        global default timeout setting returned by :func:`getdefaulttimeout`<br/>        is used.  If *source_address* is set it must be a tuple of (host, port)<br/>        for the socket to bind as a source address before making the connection.<br/>        A host of '' or port 0 tells the OS to use the default.<br/>        """<br/>    <br/>        host, port = address<br/>        err = None<br/>        for res in getaddrinfo(host, port, 0, SOCK_STREAM):<br/>            af, socktype, proto, canonname, sa = res<br/>            sock = None<br/>            try:<br/>                sock = socket(af, socktype, proto)<br/>                if timeout is not _GLOBAL_DEFAULT_TIMEOUT:<br/>                    sock.settimeout(timeout)<br/>                if source_address:<br/>                    sock.bind(source_address)<br/>                sock.connect(sa)<br/>                return sock<br/>    <br/>            except error as _:<br/>                err = _<br/>                if sock is not None:<br/>                    sock.close()<br/>    <br/>        if err is not None:<br/>&gt;           raise err<br/><span class="error">E           error: [Errno 111] Connection refused</span><br/><br/>/usr/lib/python2.7/socket.py:575: error<br/></div></td></tr></tbody>
      <tbody class="error results-table-row">
        <tr>
          <td class="col-result">Error</td>
          <td class="col-name">tests/test_arm_controls.py::test_ee_xyz_control[True]::setup</td>
          <td class="col-duration">0.00</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">@pytest.fixture(scope="module")<br/>    def create_robot():<br/>&gt;       return Robot('locobot', use_base=False)<br/><br/>test_arm_controls.py:17: <br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/core.py:105: in __init__<br/>    self.arm = arm_class(self.configs, **arm_config)<br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/locobot/arm.py:40: in __init__<br/>    use_arm = rospy.get_param('use_arm', False)<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/client.py:465: in get_param<br/>    return _param_server[param_name] #MasterProxy does all the magic for us<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/msproxy.py:121: in __getitem__<br/>    code, msg, value = self.target.getParam(rospy.names.get_caller_id(), resolved_key)<br/>/usr/lib/python2.7/xmlrpclib.py:1243: in __call__<br/>    return self.__send(self.__name, args)<br/>/usr/lib/python2.7/xmlrpclib.py:1602: in __request<br/>    verbose=self.__verbose<br/>/usr/lib/python2.7/xmlrpclib.py:1283: in request<br/>    return self.single_request(host, handler, request_body, verbose)<br/>/usr/lib/python2.7/xmlrpclib.py:1311: in single_request<br/>    self.send_content(h, request_body)<br/>/usr/lib/python2.7/xmlrpclib.py:1459: in send_content<br/>    connection.endheaders(request_body)<br/>/usr/lib/python2.7/httplib.py:1053: in endheaders<br/>    self._send_output(message_body)<br/>/usr/lib/python2.7/httplib.py:897: in _send_output<br/>    self.send(msg)<br/>/usr/lib/python2.7/httplib.py:859: in send<br/>    self.connect()<br/>/usr/lib/python2.7/httplib.py:836: in connect<br/>    self.timeout, self.source_address)<br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/><br/>address = ('localhost', 11311), timeout = &lt;object object at 0x7f5b8b5b83b0&gt;, source_address = None<br/><br/>    def create_connection(address, timeout=_GLOBAL_DEFAULT_TIMEOUT,<br/>                          source_address=None):<br/>        """Connect to *address* and return the socket object.<br/>    <br/>        Convenience function.  Connect to *address* (a 2-tuple ``(host,<br/>        port)``) and return the socket object.  Passing the optional<br/>        *timeout* parameter will set the timeout on the socket instance<br/>        before attempting to connect.  If no *timeout* is supplied, the<br/>        global default timeout setting returned by :func:`getdefaulttimeout`<br/>        is used.  If *source_address* is set it must be a tuple of (host, port)<br/>        for the socket to bind as a source address before making the connection.<br/>        A host of '' or port 0 tells the OS to use the default.<br/>        """<br/>    <br/>        host, port = address<br/>        err = None<br/>        for res in getaddrinfo(host, port, 0, SOCK_STREAM):<br/>            af, socktype, proto, canonname, sa = res<br/>            sock = None<br/>            try:<br/>                sock = socket(af, socktype, proto)<br/>                if timeout is not _GLOBAL_DEFAULT_TIMEOUT:<br/>                    sock.settimeout(timeout)<br/>                if source_address:<br/>                    sock.bind(source_address)<br/>                sock.connect(sa)<br/>                return sock<br/>    <br/>            except error as _:<br/>                err = _<br/>                if sock is not None:<br/>                    sock.close()<br/>    <br/>        if err is not None:<br/>&gt;           raise err<br/><span class="error">E           error: [Errno 111] Connection refused</span><br/><br/>/usr/lib/python2.7/socket.py:575: error<br/></div></td></tr></tbody>
      <tbody class="error results-table-row">
        <tr>
          <td class="col-result">Error</td>
          <td class="col-name">tests/test_arm_controls.py::test_ee_xyz_control[False]::setup</td>
          <td class="col-duration">0.00</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">@pytest.fixture(scope="module")<br/>    def create_robot():<br/>&gt;       return Robot('locobot', use_base=False)<br/><br/>test_arm_controls.py:17: <br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/core.py:105: in __init__<br/>    self.arm = arm_class(self.configs, **arm_config)<br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/locobot/arm.py:40: in __init__<br/>    use_arm = rospy.get_param('use_arm', False)<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/client.py:465: in get_param<br/>    return _param_server[param_name] #MasterProxy does all the magic for us<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/msproxy.py:121: in __getitem__<br/>    code, msg, value = self.target.getParam(rospy.names.get_caller_id(), resolved_key)<br/>/usr/lib/python2.7/xmlrpclib.py:1243: in __call__<br/>    return self.__send(self.__name, args)<br/>/usr/lib/python2.7/xmlrpclib.py:1602: in __request<br/>    verbose=self.__verbose<br/>/usr/lib/python2.7/xmlrpclib.py:1283: in request<br/>    return self.single_request(host, handler, request_body, verbose)<br/>/usr/lib/python2.7/xmlrpclib.py:1311: in single_request<br/>    self.send_content(h, request_body)<br/>/usr/lib/python2.7/xmlrpclib.py:1459: in send_content<br/>    connection.endheaders(request_body)<br/>/usr/lib/python2.7/httplib.py:1053: in endheaders<br/>    self._send_output(message_body)<br/>/usr/lib/python2.7/httplib.py:897: in _send_output<br/>    self.send(msg)<br/>/usr/lib/python2.7/httplib.py:859: in send<br/>    self.connect()<br/>/usr/lib/python2.7/httplib.py:836: in connect<br/>    self.timeout, self.source_address)<br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/><br/>address = ('localhost', 11311), timeout = &lt;object object at 0x7f5b8b5b83b0&gt;, source_address = None<br/><br/>    def create_connection(address, timeout=_GLOBAL_DEFAULT_TIMEOUT,<br/>                          source_address=None):<br/>        """Connect to *address* and return the socket object.<br/>    <br/>        Convenience function.  Connect to *address* (a 2-tuple ``(host,<br/>        port)``) and return the socket object.  Passing the optional<br/>        *timeout* parameter will set the timeout on the socket instance<br/>        before attempting to connect.  If no *timeout* is supplied, the<br/>        global default timeout setting returned by :func:`getdefaulttimeout`<br/>        is used.  If *source_address* is set it must be a tuple of (host, port)<br/>        for the socket to bind as a source address before making the connection.<br/>        A host of '' or port 0 tells the OS to use the default.<br/>        """<br/>    <br/>        host, port = address<br/>        err = None<br/>        for res in getaddrinfo(host, port, 0, SOCK_STREAM):<br/>            af, socktype, proto, canonname, sa = res<br/>            sock = None<br/>            try:<br/>                sock = socket(af, socktype, proto)<br/>                if timeout is not _GLOBAL_DEFAULT_TIMEOUT:<br/>                    sock.settimeout(timeout)<br/>                if source_address:<br/>                    sock.bind(source_address)<br/>                sock.connect(sa)<br/>                return sock<br/>    <br/>            except error as _:<br/>                err = _<br/>                if sock is not None:<br/>                    sock.close()<br/>    <br/>        if err is not None:<br/>&gt;           raise err<br/><span class="error">E           error: [Errno 111] Connection refused</span><br/><br/>/usr/lib/python2.7/socket.py:575: error<br/></div></td></tr></tbody>
      <tbody class="error results-table-row">
        <tr>
          <td class="col-result">Error</td>
          <td class="col-name">tests/test_arm_controls.py::test_gripper_control::setup</td>
          <td class="col-duration">0.00</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">@pytest.fixture(scope="module")<br/>    def create_robot():<br/>&gt;       return Robot('locobot', use_base=False)<br/><br/>test_arm_controls.py:17: <br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/core.py:105: in __init__<br/>    self.arm = arm_class(self.configs, **arm_config)<br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/locobot/arm.py:40: in __init__<br/>    use_arm = rospy.get_param('use_arm', False)<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/client.py:465: in get_param<br/>    return _param_server[param_name] #MasterProxy does all the magic for us<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/msproxy.py:121: in __getitem__<br/>    code, msg, value = self.target.getParam(rospy.names.get_caller_id(), resolved_key)<br/>/usr/lib/python2.7/xmlrpclib.py:1243: in __call__<br/>    return self.__send(self.__name, args)<br/>/usr/lib/python2.7/xmlrpclib.py:1602: in __request<br/>    verbose=self.__verbose<br/>/usr/lib/python2.7/xmlrpclib.py:1283: in request<br/>    return self.single_request(host, handler, request_body, verbose)<br/>/usr/lib/python2.7/xmlrpclib.py:1311: in single_request<br/>    self.send_content(h, request_body)<br/>/usr/lib/python2.7/xmlrpclib.py:1459: in send_content<br/>    connection.endheaders(request_body)<br/>/usr/lib/python2.7/httplib.py:1053: in endheaders<br/>    self._send_output(message_body)<br/>/usr/lib/python2.7/httplib.py:897: in _send_output<br/>    self.send(msg)<br/>/usr/lib/python2.7/httplib.py:859: in send<br/>    self.connect()<br/>/usr/lib/python2.7/httplib.py:836: in connect<br/>    self.timeout, self.source_address)<br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/><br/>address = ('localhost', 11311), timeout = &lt;object object at 0x7f5b8b5b83b0&gt;, source_address = None<br/><br/>    def create_connection(address, timeout=_GLOBAL_DEFAULT_TIMEOUT,<br/>                          source_address=None):<br/>        """Connect to *address* and return the socket object.<br/>    <br/>        Convenience function.  Connect to *address* (a 2-tuple ``(host,<br/>        port)``) and return the socket object.  Passing the optional<br/>        *timeout* parameter will set the timeout on the socket instance<br/>        before attempting to connect.  If no *timeout* is supplied, the<br/>        global default timeout setting returned by :func:`getdefaulttimeout`<br/>        is used.  If *source_address* is set it must be a tuple of (host, port)<br/>        for the socket to bind as a source address before making the connection.<br/>        A host of '' or port 0 tells the OS to use the default.<br/>        """<br/>    <br/>        host, port = address<br/>        err = None<br/>        for res in getaddrinfo(host, port, 0, SOCK_STREAM):<br/>            af, socktype, proto, canonname, sa = res<br/>            sock = None<br/>            try:<br/>                sock = socket(af, socktype, proto)<br/>                if timeout is not _GLOBAL_DEFAULT_TIMEOUT:<br/>                    sock.settimeout(timeout)<br/>                if source_address:<br/>                    sock.bind(source_address)<br/>                sock.connect(sa)<br/>                return sock<br/>    <br/>            except error as _:<br/>                err = _<br/>                if sock is not None:<br/>                    sock.close()<br/>    <br/>        if err is not None:<br/>&gt;           raise err<br/><span class="error">E           error: [Errno 111] Connection refused</span><br/><br/>/usr/lib/python2.7/socket.py:575: error<br/></div></td></tr></tbody>
      <tbody class="error results-table-row">
        <tr>
          <td class="col-result">Error</td>
          <td class="col-name">tests/test_camera.py::test_position_control[target_position0]::setup</td>
          <td class="col-duration">0.00</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">@pytest.fixture(scope="module")<br/>    def create_robot():<br/>        return Robot('locobot', use_camera=True,<br/>                     use_base=False,<br/>                     use_arm=False,<br/>&gt;                    use_gripper=False)<br/><br/>test_camera.py:17: <br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/core.py:113: in __init__<br/>    self.camera = camera_class(self.configs, **camera_config)<br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/locobot/camera.py:272: in __init__<br/>    use_camera = rospy.get_param('use_camera', False)<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/client.py:465: in get_param<br/>    return _param_server[param_name] #MasterProxy does all the magic for us<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/msproxy.py:121: in __getitem__<br/>    code, msg, value = self.target.getParam(rospy.names.get_caller_id(), resolved_key)<br/>/usr/lib/python2.7/xmlrpclib.py:1243: in __call__<br/>    return self.__send(self.__name, args)<br/>/usr/lib/python2.7/xmlrpclib.py:1602: in __request<br/>    verbose=self.__verbose<br/>/usr/lib/python2.7/xmlrpclib.py:1283: in request<br/>    return self.single_request(host, handler, request_body, verbose)<br/>/usr/lib/python2.7/xmlrpclib.py:1311: in single_request<br/>    self.send_content(h, request_body)<br/>/usr/lib/python2.7/xmlrpclib.py:1459: in send_content<br/>    connection.endheaders(request_body)<br/>/usr/lib/python2.7/httplib.py:1053: in endheaders<br/>    self._send_output(message_body)<br/>/usr/lib/python2.7/httplib.py:897: in _send_output<br/>    self.send(msg)<br/>/usr/lib/python2.7/httplib.py:859: in send<br/>    self.connect()<br/>/usr/lib/python2.7/httplib.py:836: in connect<br/>    self.timeout, self.source_address)<br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/><br/>address = ('localhost', 11311), timeout = &lt;object object at 0x7f5b8b5b83b0&gt;, source_address = None<br/><br/>    def create_connection(address, timeout=_GLOBAL_DEFAULT_TIMEOUT,<br/>                          source_address=None):<br/>        """Connect to *address* and return the socket object.<br/>    <br/>        Convenience function.  Connect to *address* (a 2-tuple ``(host,<br/>        port)``) and return the socket object.  Passing the optional<br/>        *timeout* parameter will set the timeout on the socket instance<br/>        before attempting to connect.  If no *timeout* is supplied, the<br/>        global default timeout setting returned by :func:`getdefaulttimeout`<br/>        is used.  If *source_address* is set it must be a tuple of (host, port)<br/>        for the socket to bind as a source address before making the connection.<br/>        A host of '' or port 0 tells the OS to use the default.<br/>        """<br/>    <br/>        host, port = address<br/>        err = None<br/>        for res in getaddrinfo(host, port, 0, SOCK_STREAM):<br/>            af, socktype, proto, canonname, sa = res<br/>            sock = None<br/>            try:<br/>                sock = socket(af, socktype, proto)<br/>                if timeout is not _GLOBAL_DEFAULT_TIMEOUT:<br/>                    sock.settimeout(timeout)<br/>                if source_address:<br/>                    sock.bind(source_address)<br/>                sock.connect(sa)<br/>                return sock<br/>    <br/>            except error as _:<br/>                err = _<br/>                if sock is not None:<br/>                    sock.close()<br/>    <br/>        if err is not None:<br/>&gt;           raise err<br/><span class="error">E           error: [Errno 111] Connection refused</span><br/><br/>/usr/lib/python2.7/socket.py:575: error<br/>---------------------------- Captured stdout setup -----------------------------<br/>['husky', 'locobot', 'sawyer', 'turtlebot']
</div></td></tr></tbody>
      <tbody class="error results-table-row">
        <tr>
          <td class="col-result">Error</td>
          <td class="col-name">tests/test_camera.py::test_position_control[target_position1]::setup</td>
          <td class="col-duration">0.00</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">@pytest.fixture(scope="module")<br/>    def create_robot():<br/>        return Robot('locobot', use_camera=True,<br/>                     use_base=False,<br/>                     use_arm=False,<br/>&gt;                    use_gripper=False)<br/><br/>test_camera.py:17: <br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/core.py:113: in __init__<br/>    self.camera = camera_class(self.configs, **camera_config)<br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/locobot/camera.py:272: in __init__<br/>    use_camera = rospy.get_param('use_camera', False)<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/client.py:465: in get_param<br/>    return _param_server[param_name] #MasterProxy does all the magic for us<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/msproxy.py:121: in __getitem__<br/>    code, msg, value = self.target.getParam(rospy.names.get_caller_id(), resolved_key)<br/>/usr/lib/python2.7/xmlrpclib.py:1243: in __call__<br/>    return self.__send(self.__name, args)<br/>/usr/lib/python2.7/xmlrpclib.py:1602: in __request<br/>    verbose=self.__verbose<br/>/usr/lib/python2.7/xmlrpclib.py:1283: in request<br/>    return self.single_request(host, handler, request_body, verbose)<br/>/usr/lib/python2.7/xmlrpclib.py:1311: in single_request<br/>    self.send_content(h, request_body)<br/>/usr/lib/python2.7/xmlrpclib.py:1459: in send_content<br/>    connection.endheaders(request_body)<br/>/usr/lib/python2.7/httplib.py:1053: in endheaders<br/>    self._send_output(message_body)<br/>/usr/lib/python2.7/httplib.py:897: in _send_output<br/>    self.send(msg)<br/>/usr/lib/python2.7/httplib.py:859: in send<br/>    self.connect()<br/>/usr/lib/python2.7/httplib.py:836: in connect<br/>    self.timeout, self.source_address)<br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/><br/>address = ('localhost', 11311), timeout = &lt;object object at 0x7f5b8b5b83b0&gt;, source_address = None<br/><br/>    def create_connection(address, timeout=_GLOBAL_DEFAULT_TIMEOUT,<br/>                          source_address=None):<br/>        """Connect to *address* and return the socket object.<br/>    <br/>        Convenience function.  Connect to *address* (a 2-tuple ``(host,<br/>        port)``) and return the socket object.  Passing the optional<br/>        *timeout* parameter will set the timeout on the socket instance<br/>        before attempting to connect.  If no *timeout* is supplied, the<br/>        global default timeout setting returned by :func:`getdefaulttimeout`<br/>        is used.  If *source_address* is set it must be a tuple of (host, port)<br/>        for the socket to bind as a source address before making the connection.<br/>        A host of '' or port 0 tells the OS to use the default.<br/>        """<br/>    <br/>        host, port = address<br/>        err = None<br/>        for res in getaddrinfo(host, port, 0, SOCK_STREAM):<br/>            af, socktype, proto, canonname, sa = res<br/>            sock = None<br/>            try:<br/>                sock = socket(af, socktype, proto)<br/>                if timeout is not _GLOBAL_DEFAULT_TIMEOUT:<br/>                    sock.settimeout(timeout)<br/>                if source_address:<br/>                    sock.bind(source_address)<br/>                sock.connect(sa)<br/>                return sock<br/>    <br/>            except error as _:<br/>                err = _<br/>                if sock is not None:<br/>                    sock.close()<br/>    <br/>        if err is not None:<br/>&gt;           raise err<br/><span class="error">E           error: [Errno 111] Connection refused</span><br/><br/>/usr/lib/python2.7/socket.py:575: error<br/></div></td></tr></tbody>
      <tbody class="error results-table-row">
        <tr>
          <td class="col-result">Error</td>
          <td class="col-name">tests/test_camera.py::test_position_control[target_position2]::setup</td>
          <td class="col-duration">0.00</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">@pytest.fixture(scope="module")<br/>    def create_robot():<br/>        return Robot('locobot', use_camera=True,<br/>                     use_base=False,<br/>                     use_arm=False,<br/>&gt;                    use_gripper=False)<br/><br/>test_camera.py:17: <br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/core.py:113: in __init__<br/>    self.camera = camera_class(self.configs, **camera_config)<br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/locobot/camera.py:272: in __init__<br/>    use_camera = rospy.get_param('use_camera', False)<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/client.py:465: in get_param<br/>    return _param_server[param_name] #MasterProxy does all the magic for us<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/msproxy.py:121: in __getitem__<br/>    code, msg, value = self.target.getParam(rospy.names.get_caller_id(), resolved_key)<br/>/usr/lib/python2.7/xmlrpclib.py:1243: in __call__<br/>    return self.__send(self.__name, args)<br/>/usr/lib/python2.7/xmlrpclib.py:1602: in __request<br/>    verbose=self.__verbose<br/>/usr/lib/python2.7/xmlrpclib.py:1283: in request<br/>    return self.single_request(host, handler, request_body, verbose)<br/>/usr/lib/python2.7/xmlrpclib.py:1311: in single_request<br/>    self.send_content(h, request_body)<br/>/usr/lib/python2.7/xmlrpclib.py:1459: in send_content<br/>    connection.endheaders(request_body)<br/>/usr/lib/python2.7/httplib.py:1053: in endheaders<br/>    self._send_output(message_body)<br/>/usr/lib/python2.7/httplib.py:897: in _send_output<br/>    self.send(msg)<br/>/usr/lib/python2.7/httplib.py:859: in send<br/>    self.connect()<br/>/usr/lib/python2.7/httplib.py:836: in connect<br/>    self.timeout, self.source_address)<br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/><br/>address = ('localhost', 11311), timeout = &lt;object object at 0x7f5b8b5b83b0&gt;, source_address = None<br/><br/>    def create_connection(address, timeout=_GLOBAL_DEFAULT_TIMEOUT,<br/>                          source_address=None):<br/>        """Connect to *address* and return the socket object.<br/>    <br/>        Convenience function.  Connect to *address* (a 2-tuple ``(host,<br/>        port)``) and return the socket object.  Passing the optional<br/>        *timeout* parameter will set the timeout on the socket instance<br/>        before attempting to connect.  If no *timeout* is supplied, the<br/>        global default timeout setting returned by :func:`getdefaulttimeout`<br/>        is used.  If *source_address* is set it must be a tuple of (host, port)<br/>        for the socket to bind as a source address before making the connection.<br/>        A host of '' or port 0 tells the OS to use the default.<br/>        """<br/>    <br/>        host, port = address<br/>        err = None<br/>        for res in getaddrinfo(host, port, 0, SOCK_STREAM):<br/>            af, socktype, proto, canonname, sa = res<br/>            sock = None<br/>            try:<br/>                sock = socket(af, socktype, proto)<br/>                if timeout is not _GLOBAL_DEFAULT_TIMEOUT:<br/>                    sock.settimeout(timeout)<br/>                if source_address:<br/>                    sock.bind(source_address)<br/>                sock.connect(sa)<br/>                return sock<br/>    <br/>            except error as _:<br/>                err = _<br/>                if sock is not None:<br/>                    sock.close()<br/>    <br/>        if err is not None:<br/>&gt;           raise err<br/><span class="error">E           error: [Errno 111] Connection refused</span><br/><br/>/usr/lib/python2.7/socket.py:575: error<br/></div></td></tr></tbody>
      <tbody class="error results-table-row">
        <tr>
          <td class="col-result">Error</td>
          <td class="col-name">tests/test_camera.py::test_position_control[target_position3]::setup</td>
          <td class="col-duration">0.00</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">@pytest.fixture(scope="module")<br/>    def create_robot():<br/>        return Robot('locobot', use_camera=True,<br/>                     use_base=False,<br/>                     use_arm=False,<br/>&gt;                    use_gripper=False)<br/><br/>test_camera.py:17: <br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/core.py:113: in __init__<br/>    self.camera = camera_class(self.configs, **camera_config)<br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/locobot/camera.py:272: in __init__<br/>    use_camera = rospy.get_param('use_camera', False)<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/client.py:465: in get_param<br/>    return _param_server[param_name] #MasterProxy does all the magic for us<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/msproxy.py:121: in __getitem__<br/>    code, msg, value = self.target.getParam(rospy.names.get_caller_id(), resolved_key)<br/>/usr/lib/python2.7/xmlrpclib.py:1243: in __call__<br/>    return self.__send(self.__name, args)<br/>/usr/lib/python2.7/xmlrpclib.py:1602: in __request<br/>    verbose=self.__verbose<br/>/usr/lib/python2.7/xmlrpclib.py:1283: in request<br/>    return self.single_request(host, handler, request_body, verbose)<br/>/usr/lib/python2.7/xmlrpclib.py:1311: in single_request<br/>    self.send_content(h, request_body)<br/>/usr/lib/python2.7/xmlrpclib.py:1459: in send_content<br/>    connection.endheaders(request_body)<br/>/usr/lib/python2.7/httplib.py:1053: in endheaders<br/>    self._send_output(message_body)<br/>/usr/lib/python2.7/httplib.py:897: in _send_output<br/>    self.send(msg)<br/>/usr/lib/python2.7/httplib.py:859: in send<br/>    self.connect()<br/>/usr/lib/python2.7/httplib.py:836: in connect<br/>    self.timeout, self.source_address)<br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/><br/>address = ('localhost', 11311), timeout = &lt;object object at 0x7f5b8b5b83b0&gt;, source_address = None<br/><br/>    def create_connection(address, timeout=_GLOBAL_DEFAULT_TIMEOUT,<br/>                          source_address=None):<br/>        """Connect to *address* and return the socket object.<br/>    <br/>        Convenience function.  Connect to *address* (a 2-tuple ``(host,<br/>        port)``) and return the socket object.  Passing the optional<br/>        *timeout* parameter will set the timeout on the socket instance<br/>        before attempting to connect.  If no *timeout* is supplied, the<br/>        global default timeout setting returned by :func:`getdefaulttimeout`<br/>        is used.  If *source_address* is set it must be a tuple of (host, port)<br/>        for the socket to bind as a source address before making the connection.<br/>        A host of '' or port 0 tells the OS to use the default.<br/>        """<br/>    <br/>        host, port = address<br/>        err = None<br/>        for res in getaddrinfo(host, port, 0, SOCK_STREAM):<br/>            af, socktype, proto, canonname, sa = res<br/>            sock = None<br/>            try:<br/>                sock = socket(af, socktype, proto)<br/>                if timeout is not _GLOBAL_DEFAULT_TIMEOUT:<br/>                    sock.settimeout(timeout)<br/>                if source_address:<br/>                    sock.bind(source_address)<br/>                sock.connect(sa)<br/>                return sock<br/>    <br/>            except error as _:<br/>                err = _<br/>                if sock is not None:<br/>                    sock.close()<br/>    <br/>        if err is not None:<br/>&gt;           raise err<br/><span class="error">E           error: [Errno 111] Connection refused</span><br/><br/>/usr/lib/python2.7/socket.py:575: error<br/></div></td></tr></tbody>
      <tbody class="error results-table-row">
        <tr>
          <td class="col-result">Error</td>
          <td class="col-name">tests/test_camera.py::test_position_control[target_position4]::setup</td>
          <td class="col-duration">0.00</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">@pytest.fixture(scope="module")<br/>    def create_robot():<br/>        return Robot('locobot', use_camera=True,<br/>                     use_base=False,<br/>                     use_arm=False,<br/>&gt;                    use_gripper=False)<br/><br/>test_camera.py:17: <br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/core.py:113: in __init__<br/>    self.camera = camera_class(self.configs, **camera_config)<br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/locobot/camera.py:272: in __init__<br/>    use_camera = rospy.get_param('use_camera', False)<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/client.py:465: in get_param<br/>    return _param_server[param_name] #MasterProxy does all the magic for us<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/msproxy.py:121: in __getitem__<br/>    code, msg, value = self.target.getParam(rospy.names.get_caller_id(), resolved_key)<br/>/usr/lib/python2.7/xmlrpclib.py:1243: in __call__<br/>    return self.__send(self.__name, args)<br/>/usr/lib/python2.7/xmlrpclib.py:1602: in __request<br/>    verbose=self.__verbose<br/>/usr/lib/python2.7/xmlrpclib.py:1283: in request<br/>    return self.single_request(host, handler, request_body, verbose)<br/>/usr/lib/python2.7/xmlrpclib.py:1311: in single_request<br/>    self.send_content(h, request_body)<br/>/usr/lib/python2.7/xmlrpclib.py:1459: in send_content<br/>    connection.endheaders(request_body)<br/>/usr/lib/python2.7/httplib.py:1053: in endheaders<br/>    self._send_output(message_body)<br/>/usr/lib/python2.7/httplib.py:897: in _send_output<br/>    self.send(msg)<br/>/usr/lib/python2.7/httplib.py:859: in send<br/>    self.connect()<br/>/usr/lib/python2.7/httplib.py:836: in connect<br/>    self.timeout, self.source_address)<br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/><br/>address = ('localhost', 11311), timeout = &lt;object object at 0x7f5b8b5b83b0&gt;, source_address = None<br/><br/>    def create_connection(address, timeout=_GLOBAL_DEFAULT_TIMEOUT,<br/>                          source_address=None):<br/>        """Connect to *address* and return the socket object.<br/>    <br/>        Convenience function.  Connect to *address* (a 2-tuple ``(host,<br/>        port)``) and return the socket object.  Passing the optional<br/>        *timeout* parameter will set the timeout on the socket instance<br/>        before attempting to connect.  If no *timeout* is supplied, the<br/>        global default timeout setting returned by :func:`getdefaulttimeout`<br/>        is used.  If *source_address* is set it must be a tuple of (host, port)<br/>        for the socket to bind as a source address before making the connection.<br/>        A host of '' or port 0 tells the OS to use the default.<br/>        """<br/>    <br/>        host, port = address<br/>        err = None<br/>        for res in getaddrinfo(host, port, 0, SOCK_STREAM):<br/>            af, socktype, proto, canonname, sa = res<br/>            sock = None<br/>            try:<br/>                sock = socket(af, socktype, proto)<br/>                if timeout is not _GLOBAL_DEFAULT_TIMEOUT:<br/>                    sock.settimeout(timeout)<br/>                if source_address:<br/>                    sock.bind(source_address)<br/>                sock.connect(sa)<br/>                return sock<br/>    <br/>            except error as _:<br/>                err = _<br/>                if sock is not None:<br/>                    sock.close()<br/>    <br/>        if err is not None:<br/>&gt;           raise err<br/><span class="error">E           error: [Errno 111] Connection refused</span><br/><br/>/usr/lib/python2.7/socket.py:575: error<br/></div></td></tr></tbody>
      <tbody class="error results-table-row">
        <tr>
          <td class="col-result">Error</td>
          <td class="col-name">tests/test_camera.py::test_get_images::setup</td>
          <td class="col-duration">0.00</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">@pytest.fixture(scope="module")<br/>    def create_robot():<br/>        return Robot('locobot', use_camera=True,<br/>                     use_base=False,<br/>                     use_arm=False,<br/>&gt;                    use_gripper=False)<br/><br/>test_camera.py:17: <br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/core.py:113: in __init__<br/>    self.camera = camera_class(self.configs, **camera_config)<br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/locobot/camera.py:272: in __init__<br/>    use_camera = rospy.get_param('use_camera', False)<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/client.py:465: in get_param<br/>    return _param_server[param_name] #MasterProxy does all the magic for us<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/msproxy.py:121: in __getitem__<br/>    code, msg, value = self.target.getParam(rospy.names.get_caller_id(), resolved_key)<br/>/usr/lib/python2.7/xmlrpclib.py:1243: in __call__<br/>    return self.__send(self.__name, args)<br/>/usr/lib/python2.7/xmlrpclib.py:1602: in __request<br/>    verbose=self.__verbose<br/>/usr/lib/python2.7/xmlrpclib.py:1283: in request<br/>    return self.single_request(host, handler, request_body, verbose)<br/>/usr/lib/python2.7/xmlrpclib.py:1311: in single_request<br/>    self.send_content(h, request_body)<br/>/usr/lib/python2.7/xmlrpclib.py:1459: in send_content<br/>    connection.endheaders(request_body)<br/>/usr/lib/python2.7/httplib.py:1053: in endheaders<br/>    self._send_output(message_body)<br/>/usr/lib/python2.7/httplib.py:897: in _send_output<br/>    self.send(msg)<br/>/usr/lib/python2.7/httplib.py:859: in send<br/>    self.connect()<br/>/usr/lib/python2.7/httplib.py:836: in connect<br/>    self.timeout, self.source_address)<br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/><br/>address = ('localhost', 11311), timeout = &lt;object object at 0x7f5b8b5b83b0&gt;, source_address = None<br/><br/>    def create_connection(address, timeout=_GLOBAL_DEFAULT_TIMEOUT,<br/>                          source_address=None):<br/>        """Connect to *address* and return the socket object.<br/>    <br/>        Convenience function.  Connect to *address* (a 2-tuple ``(host,<br/>        port)``) and return the socket object.  Passing the optional<br/>        *timeout* parameter will set the timeout on the socket instance<br/>        before attempting to connect.  If no *timeout* is supplied, the<br/>        global default timeout setting returned by :func:`getdefaulttimeout`<br/>        is used.  If *source_address* is set it must be a tuple of (host, port)<br/>        for the socket to bind as a source address before making the connection.<br/>        A host of '' or port 0 tells the OS to use the default.<br/>        """<br/>    <br/>        host, port = address<br/>        err = None<br/>        for res in getaddrinfo(host, port, 0, SOCK_STREAM):<br/>            af, socktype, proto, canonname, sa = res<br/>            sock = None<br/>            try:<br/>                sock = socket(af, socktype, proto)<br/>                if timeout is not _GLOBAL_DEFAULT_TIMEOUT:<br/>                    sock.settimeout(timeout)<br/>                if source_address:<br/>                    sock.bind(source_address)<br/>                sock.connect(sa)<br/>                return sock<br/>    <br/>            except error as _:<br/>                err = _<br/>                if sock is not None:<br/>                    sock.close()<br/>    <br/>        if err is not None:<br/>&gt;           raise err<br/><span class="error">E           error: [Errno 111] Connection refused</span><br/><br/>/usr/lib/python2.7/socket.py:575: error<br/></div></td></tr></tbody>
      <tbody class="failed results-table-row">
        <tr>
          <td class="col-result">Failed</td>
          <td class="col-name">tests/test_make_robots.py::test_make_robot_base[movebase-ilqr]</td>
          <td class="col-duration">23.47</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">botname = 'locobot', base_controller = 'ilqr', base_planner = 'movebase'<br/><br/>    @pytest.mark.parametrize("base_controller", ['ilqr', 'proportional', 'movebase'])<br/>    @pytest.mark.parametrize("base_planner", ['movebase', 'none'])<br/>    def test_make_robot_base(botname, base_controller, base_planner):<br/>        bot = Robot(botname, base_config={'base_controller': base_controller,<br/>                                          'base_planner': base_planner},<br/>&gt;                   use_arm=False, use_camera=False,use_gripper=False)<br/><br/>test_make_robots.py:22: <br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/core.py:109: in __init__<br/>    self.base = base_class(self.configs, **base_config)<br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/locobot/base.py:204: in __init__<br/>    'use_base', False) or rospy.get_param(<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/client.py:465: in get_param<br/>    return _param_server[param_name] #MasterProxy does all the magic for us<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/msproxy.py:121: in __getitem__<br/>    code, msg, value = self.target.getParam(rospy.names.get_caller_id(), resolved_key)<br/>/usr/lib/python2.7/xmlrpclib.py:1243: in __call__<br/>    return self.__send(self.__name, args)<br/>/usr/lib/python2.7/xmlrpclib.py:1602: in __request<br/>    verbose=self.__verbose<br/>/usr/lib/python2.7/xmlrpclib.py:1283: in request<br/>    return self.single_request(host, handler, request_body, verbose)<br/>/usr/lib/python2.7/xmlrpclib.py:1311: in single_request<br/>    self.send_content(h, request_body)<br/>/usr/lib/python2.7/xmlrpclib.py:1459: in send_content<br/>    connection.endheaders(request_body)<br/>/usr/lib/python2.7/httplib.py:1053: in endheaders<br/>    self._send_output(message_body)<br/>/usr/lib/python2.7/httplib.py:897: in _send_output<br/>    self.send(msg)<br/>/usr/lib/python2.7/httplib.py:859: in send<br/>    self.connect()<br/>/usr/lib/python2.7/httplib.py:836: in connect<br/>    self.timeout, self.source_address)<br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/><br/>address = ('localhost', 11311), timeout = &lt;object object at 0x7f5b8b5b83b0&gt;, source_address = None<br/><br/>    def create_connection(address, timeout=_GLOBAL_DEFAULT_TIMEOUT,<br/>                          source_address=None):<br/>        """Connect to *address* and return the socket object.<br/>    <br/>        Convenience function.  Connect to *address* (a 2-tuple ``(host,<br/>        port)``) and return the socket object.  Passing the optional<br/>        *timeout* parameter will set the timeout on the socket instance<br/>        before attempting to connect.  If no *timeout* is supplied, the<br/>        global default timeout setting returned by :func:`getdefaulttimeout`<br/>        is used.  If *source_address* is set it must be a tuple of (host, port)<br/>        for the socket to bind as a source address before making the connection.<br/>        A host of '' or port 0 tells the OS to use the default.<br/>        """<br/>    <br/>        host, port = address<br/>        err = None<br/>        for res in getaddrinfo(host, port, 0, SOCK_STREAM):<br/>            af, socktype, proto, canonname, sa = res<br/>            sock = None<br/>            try:<br/>                sock = socket(af, socktype, proto)<br/>                if timeout is not _GLOBAL_DEFAULT_TIMEOUT:<br/>                    sock.settimeout(timeout)<br/>                if source_address:<br/>                    sock.bind(source_address)<br/>                sock.connect(sa)<br/>                return sock<br/>    <br/>            except error as _:<br/>                err = _<br/>                if sock is not None:<br/>                    sock.close()<br/>    <br/>        if err is not None:<br/>&gt;           raise err<br/><span class="error">E           error: [Errno 111] Connection refused</span><br/><br/>/usr/lib/python2.7/socket.py:575: error<br/>----------------------------- Captured stdout call -----------------------------<br/>['husky', 'locobot', 'sawyer', 'turtlebot']
Unable to register with master node [http://localhost:11311]: master may not be running yet. Will keep trying.
----------------------------- Captured stderr call -----------------------------<br/>[WARN] [1561556607.431675]: ROS node [pyrobot] has already been initialized
</div></td></tr></tbody>
      <tbody class="failed results-table-row">
        <tr>
          <td class="col-result">Failed</td>
          <td class="col-name">tests/test_make_robots.py::test_make_robot_base[movebase-proportional]</td>
          <td class="col-duration">0.00</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">botname = 'locobot', base_controller = 'proportional', base_planner = 'movebase'<br/><br/>    @pytest.mark.parametrize("base_controller", ['ilqr', 'proportional', 'movebase'])<br/>    @pytest.mark.parametrize("base_planner", ['movebase', 'none'])<br/>    def test_make_robot_base(botname, base_controller, base_planner):<br/>        bot = Robot(botname, base_config={'base_controller': base_controller,<br/>                                          'base_planner': base_planner},<br/>&gt;                   use_arm=False, use_camera=False,use_gripper=False)<br/><br/>test_make_robots.py:22: <br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/core.py:109: in __init__<br/>    self.base = base_class(self.configs, **base_config)<br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/locobot/base.py:204: in __init__<br/>    'use_base', False) or rospy.get_param(<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/client.py:465: in get_param<br/>    return _param_server[param_name] #MasterProxy does all the magic for us<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/msproxy.py:121: in __getitem__<br/>    code, msg, value = self.target.getParam(rospy.names.get_caller_id(), resolved_key)<br/>/usr/lib/python2.7/xmlrpclib.py:1243: in __call__<br/>    return self.__send(self.__name, args)<br/>/usr/lib/python2.7/xmlrpclib.py:1602: in __request<br/>    verbose=self.__verbose<br/>/usr/lib/python2.7/xmlrpclib.py:1283: in request<br/>    return self.single_request(host, handler, request_body, verbose)<br/>/usr/lib/python2.7/xmlrpclib.py:1311: in single_request<br/>    self.send_content(h, request_body)<br/>/usr/lib/python2.7/xmlrpclib.py:1459: in send_content<br/>    connection.endheaders(request_body)<br/>/usr/lib/python2.7/httplib.py:1053: in endheaders<br/>    self._send_output(message_body)<br/>/usr/lib/python2.7/httplib.py:897: in _send_output<br/>    self.send(msg)<br/>/usr/lib/python2.7/httplib.py:859: in send<br/>    self.connect()<br/>/usr/lib/python2.7/httplib.py:836: in connect<br/>    self.timeout, self.source_address)<br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/><br/>address = ('localhost', 11311), timeout = &lt;object object at 0x7f5b8b5b83b0&gt;, source_address = None<br/><br/>    def create_connection(address, timeout=_GLOBAL_DEFAULT_TIMEOUT,<br/>                          source_address=None):<br/>        """Connect to *address* and return the socket object.<br/>    <br/>        Convenience function.  Connect to *address* (a 2-tuple ``(host,<br/>        port)``) and return the socket object.  Passing the optional<br/>        *timeout* parameter will set the timeout on the socket instance<br/>        before attempting to connect.  If no *timeout* is supplied, the<br/>        global default timeout setting returned by :func:`getdefaulttimeout`<br/>        is used.  If *source_address* is set it must be a tuple of (host, port)<br/>        for the socket to bind as a source address before making the connection.<br/>        A host of '' or port 0 tells the OS to use the default.<br/>        """<br/>    <br/>        host, port = address<br/>        err = None<br/>        for res in getaddrinfo(host, port, 0, SOCK_STREAM):<br/>            af, socktype, proto, canonname, sa = res<br/>            sock = None<br/>            try:<br/>                sock = socket(af, socktype, proto)<br/>                if timeout is not _GLOBAL_DEFAULT_TIMEOUT:<br/>                    sock.settimeout(timeout)<br/>                if source_address:<br/>                    sock.bind(source_address)<br/>                sock.connect(sa)<br/>                return sock<br/>    <br/>            except error as _:<br/>                err = _<br/>                if sock is not None:<br/>                    sock.close()<br/>    <br/>        if err is not None:<br/>&gt;           raise err<br/><span class="error">E           error: [Errno 111] Connection refused</span><br/><br/>/usr/lib/python2.7/socket.py:575: error<br/>----------------------------- Captured stdout call -----------------------------<br/>['husky', 'locobot', 'sawyer', 'turtlebot']
</div></td></tr></tbody>
      <tbody class="failed results-table-row">
        <tr>
          <td class="col-result">Failed</td>
          <td class="col-name">tests/test_make_robots.py::test_make_robot_base[movebase-movebase]</td>
          <td class="col-duration">0.00</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">botname = 'locobot', base_controller = 'movebase', base_planner = 'movebase'<br/><br/>    @pytest.mark.parametrize("base_controller", ['ilqr', 'proportional', 'movebase'])<br/>    @pytest.mark.parametrize("base_planner", ['movebase', 'none'])<br/>    def test_make_robot_base(botname, base_controller, base_planner):<br/>        bot = Robot(botname, base_config={'base_controller': base_controller,<br/>                                          'base_planner': base_planner},<br/>&gt;                   use_arm=False, use_camera=False,use_gripper=False)<br/><br/>test_make_robots.py:22: <br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/core.py:109: in __init__<br/>    self.base = base_class(self.configs, **base_config)<br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/locobot/base.py:204: in __init__<br/>    'use_base', False) or rospy.get_param(<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/client.py:465: in get_param<br/>    return _param_server[param_name] #MasterProxy does all the magic for us<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/msproxy.py:121: in __getitem__<br/>    code, msg, value = self.target.getParam(rospy.names.get_caller_id(), resolved_key)<br/>/usr/lib/python2.7/xmlrpclib.py:1243: in __call__<br/>    return self.__send(self.__name, args)<br/>/usr/lib/python2.7/xmlrpclib.py:1602: in __request<br/>    verbose=self.__verbose<br/>/usr/lib/python2.7/xmlrpclib.py:1283: in request<br/>    return self.single_request(host, handler, request_body, verbose)<br/>/usr/lib/python2.7/xmlrpclib.py:1311: in single_request<br/>    self.send_content(h, request_body)<br/>/usr/lib/python2.7/xmlrpclib.py:1459: in send_content<br/>    connection.endheaders(request_body)<br/>/usr/lib/python2.7/httplib.py:1053: in endheaders<br/>    self._send_output(message_body)<br/>/usr/lib/python2.7/httplib.py:897: in _send_output<br/>    self.send(msg)<br/>/usr/lib/python2.7/httplib.py:859: in send<br/>    self.connect()<br/>/usr/lib/python2.7/httplib.py:836: in connect<br/>    self.timeout, self.source_address)<br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/><br/>address = ('localhost', 11311), timeout = &lt;object object at 0x7f5b8b5b83b0&gt;, source_address = None<br/><br/>    def create_connection(address, timeout=_GLOBAL_DEFAULT_TIMEOUT,<br/>                          source_address=None):<br/>        """Connect to *address* and return the socket object.<br/>    <br/>        Convenience function.  Connect to *address* (a 2-tuple ``(host,<br/>        port)``) and return the socket object.  Passing the optional<br/>        *timeout* parameter will set the timeout on the socket instance<br/>        before attempting to connect.  If no *timeout* is supplied, the<br/>        global default timeout setting returned by :func:`getdefaulttimeout`<br/>        is used.  If *source_address* is set it must be a tuple of (host, port)<br/>        for the socket to bind as a source address before making the connection.<br/>        A host of '' or port 0 tells the OS to use the default.<br/>        """<br/>    <br/>        host, port = address<br/>        err = None<br/>        for res in getaddrinfo(host, port, 0, SOCK_STREAM):<br/>            af, socktype, proto, canonname, sa = res<br/>            sock = None<br/>            try:<br/>                sock = socket(af, socktype, proto)<br/>                if timeout is not _GLOBAL_DEFAULT_TIMEOUT:<br/>                    sock.settimeout(timeout)<br/>                if source_address:<br/>                    sock.bind(source_address)<br/>                sock.connect(sa)<br/>                return sock<br/>    <br/>            except error as _:<br/>                err = _<br/>                if sock is not None:<br/>                    sock.close()<br/>    <br/>        if err is not None:<br/>&gt;           raise err<br/><span class="error">E           error: [Errno 111] Connection refused</span><br/><br/>/usr/lib/python2.7/socket.py:575: error<br/>----------------------------- Captured stdout call -----------------------------<br/>['husky', 'locobot', 'sawyer', 'turtlebot']
</div></td></tr></tbody>
      <tbody class="failed results-table-row">
        <tr>
          <td class="col-result">Failed</td>
          <td class="col-name">tests/test_make_robots.py::test_make_robot_base[none-ilqr]</td>
          <td class="col-duration">0.00</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">botname = 'locobot', base_controller = 'ilqr', base_planner = 'none'<br/><br/>    @pytest.mark.parametrize("base_controller", ['ilqr', 'proportional', 'movebase'])<br/>    @pytest.mark.parametrize("base_planner", ['movebase', 'none'])<br/>    def test_make_robot_base(botname, base_controller, base_planner):<br/>        bot = Robot(botname, base_config={'base_controller': base_controller,<br/>                                          'base_planner': base_planner},<br/>&gt;                   use_arm=False, use_camera=False,use_gripper=False)<br/><br/>test_make_robots.py:22: <br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/core.py:109: in __init__<br/>    self.base = base_class(self.configs, **base_config)<br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/locobot/base.py:204: in __init__<br/>    'use_base', False) or rospy.get_param(<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/client.py:465: in get_param<br/>    return _param_server[param_name] #MasterProxy does all the magic for us<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/msproxy.py:121: in __getitem__<br/>    code, msg, value = self.target.getParam(rospy.names.get_caller_id(), resolved_key)<br/>/usr/lib/python2.7/xmlrpclib.py:1243: in __call__<br/>    return self.__send(self.__name, args)<br/>/usr/lib/python2.7/xmlrpclib.py:1602: in __request<br/>    verbose=self.__verbose<br/>/usr/lib/python2.7/xmlrpclib.py:1283: in request<br/>    return self.single_request(host, handler, request_body, verbose)<br/>/usr/lib/python2.7/xmlrpclib.py:1311: in single_request<br/>    self.send_content(h, request_body)<br/>/usr/lib/python2.7/xmlrpclib.py:1459: in send_content<br/>    connection.endheaders(request_body)<br/>/usr/lib/python2.7/httplib.py:1053: in endheaders<br/>    self._send_output(message_body)<br/>/usr/lib/python2.7/httplib.py:897: in _send_output<br/>    self.send(msg)<br/>/usr/lib/python2.7/httplib.py:859: in send<br/>    self.connect()<br/>/usr/lib/python2.7/httplib.py:836: in connect<br/>    self.timeout, self.source_address)<br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/><br/>address = ('localhost', 11311), timeout = &lt;object object at 0x7f5b8b5b83b0&gt;, source_address = None<br/><br/>    def create_connection(address, timeout=_GLOBAL_DEFAULT_TIMEOUT,<br/>                          source_address=None):<br/>        """Connect to *address* and return the socket object.<br/>    <br/>        Convenience function.  Connect to *address* (a 2-tuple ``(host,<br/>        port)``) and return the socket object.  Passing the optional<br/>        *timeout* parameter will set the timeout on the socket instance<br/>        before attempting to connect.  If no *timeout* is supplied, the<br/>        global default timeout setting returned by :func:`getdefaulttimeout`<br/>        is used.  If *source_address* is set it must be a tuple of (host, port)<br/>        for the socket to bind as a source address before making the connection.<br/>        A host of '' or port 0 tells the OS to use the default.<br/>        """<br/>    <br/>        host, port = address<br/>        err = None<br/>        for res in getaddrinfo(host, port, 0, SOCK_STREAM):<br/>            af, socktype, proto, canonname, sa = res<br/>            sock = None<br/>            try:<br/>                sock = socket(af, socktype, proto)<br/>                if timeout is not _GLOBAL_DEFAULT_TIMEOUT:<br/>                    sock.settimeout(timeout)<br/>                if source_address:<br/>                    sock.bind(source_address)<br/>                sock.connect(sa)<br/>                return sock<br/>    <br/>            except error as _:<br/>                err = _<br/>                if sock is not None:<br/>                    sock.close()<br/>    <br/>        if err is not None:<br/>&gt;           raise err<br/><span class="error">E           error: [Errno 111] Connection refused</span><br/><br/>/usr/lib/python2.7/socket.py:575: error<br/>----------------------------- Captured stdout call -----------------------------<br/>['husky', 'locobot', 'sawyer', 'turtlebot']
</div></td></tr></tbody>
      <tbody class="failed results-table-row">
        <tr>
          <td class="col-result">Failed</td>
          <td class="col-name">tests/test_make_robots.py::test_make_robot_base[none-proportional]</td>
          <td class="col-duration">0.00</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">botname = 'locobot', base_controller = 'proportional', base_planner = 'none'<br/><br/>    @pytest.mark.parametrize("base_controller", ['ilqr', 'proportional', 'movebase'])<br/>    @pytest.mark.parametrize("base_planner", ['movebase', 'none'])<br/>    def test_make_robot_base(botname, base_controller, base_planner):<br/>        bot = Robot(botname, base_config={'base_controller': base_controller,<br/>                                          'base_planner': base_planner},<br/>&gt;                   use_arm=False, use_camera=False,use_gripper=False)<br/><br/>test_make_robots.py:22: <br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/core.py:109: in __init__<br/>    self.base = base_class(self.configs, **base_config)<br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/locobot/base.py:204: in __init__<br/>    'use_base', False) or rospy.get_param(<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/client.py:465: in get_param<br/>    return _param_server[param_name] #MasterProxy does all the magic for us<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/msproxy.py:121: in __getitem__<br/>    code, msg, value = self.target.getParam(rospy.names.get_caller_id(), resolved_key)<br/>/usr/lib/python2.7/xmlrpclib.py:1243: in __call__<br/>    return self.__send(self.__name, args)<br/>/usr/lib/python2.7/xmlrpclib.py:1602: in __request<br/>    verbose=self.__verbose<br/>/usr/lib/python2.7/xmlrpclib.py:1283: in request<br/>    return self.single_request(host, handler, request_body, verbose)<br/>/usr/lib/python2.7/xmlrpclib.py:1311: in single_request<br/>    self.send_content(h, request_body)<br/>/usr/lib/python2.7/xmlrpclib.py:1459: in send_content<br/>    connection.endheaders(request_body)<br/>/usr/lib/python2.7/httplib.py:1053: in endheaders<br/>    self._send_output(message_body)<br/>/usr/lib/python2.7/httplib.py:897: in _send_output<br/>    self.send(msg)<br/>/usr/lib/python2.7/httplib.py:859: in send<br/>    self.connect()<br/>/usr/lib/python2.7/httplib.py:836: in connect<br/>    self.timeout, self.source_address)<br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/><br/>address = ('localhost', 11311), timeout = &lt;object object at 0x7f5b8b5b83b0&gt;, source_address = None<br/><br/>    def create_connection(address, timeout=_GLOBAL_DEFAULT_TIMEOUT,<br/>                          source_address=None):<br/>        """Connect to *address* and return the socket object.<br/>    <br/>        Convenience function.  Connect to *address* (a 2-tuple ``(host,<br/>        port)``) and return the socket object.  Passing the optional<br/>        *timeout* parameter will set the timeout on the socket instance<br/>        before attempting to connect.  If no *timeout* is supplied, the<br/>        global default timeout setting returned by :func:`getdefaulttimeout`<br/>        is used.  If *source_address* is set it must be a tuple of (host, port)<br/>        for the socket to bind as a source address before making the connection.<br/>        A host of '' or port 0 tells the OS to use the default.<br/>        """<br/>    <br/>        host, port = address<br/>        err = None<br/>        for res in getaddrinfo(host, port, 0, SOCK_STREAM):<br/>            af, socktype, proto, canonname, sa = res<br/>            sock = None<br/>            try:<br/>                sock = socket(af, socktype, proto)<br/>                if timeout is not _GLOBAL_DEFAULT_TIMEOUT:<br/>                    sock.settimeout(timeout)<br/>                if source_address:<br/>                    sock.bind(source_address)<br/>                sock.connect(sa)<br/>                return sock<br/>    <br/>            except error as _:<br/>                err = _<br/>                if sock is not None:<br/>                    sock.close()<br/>    <br/>        if err is not None:<br/>&gt;           raise err<br/><span class="error">E           error: [Errno 111] Connection refused</span><br/><br/>/usr/lib/python2.7/socket.py:575: error<br/>----------------------------- Captured stdout call -----------------------------<br/>['husky', 'locobot', 'sawyer', 'turtlebot']
</div></td></tr></tbody>
      <tbody class="failed results-table-row">
        <tr>
          <td class="col-result">Failed</td>
          <td class="col-name">tests/test_make_robots.py::test_make_robot_base[none-movebase]</td>
          <td class="col-duration">0.00</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">botname = 'locobot', base_controller = 'movebase', base_planner = 'none'<br/><br/>    @pytest.mark.parametrize("base_controller", ['ilqr', 'proportional', 'movebase'])<br/>    @pytest.mark.parametrize("base_planner", ['movebase', 'none'])<br/>    def test_make_robot_base(botname, base_controller, base_planner):<br/>        bot = Robot(botname, base_config={'base_controller': base_controller,<br/>                                          'base_planner': base_planner},<br/>&gt;                   use_arm=False, use_camera=False,use_gripper=False)<br/><br/>test_make_robots.py:22: <br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/core.py:109: in __init__<br/>    self.base = base_class(self.configs, **base_config)<br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/locobot/base.py:204: in __init__<br/>    'use_base', False) or rospy.get_param(<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/client.py:465: in get_param<br/>    return _param_server[param_name] #MasterProxy does all the magic for us<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/msproxy.py:121: in __getitem__<br/>    code, msg, value = self.target.getParam(rospy.names.get_caller_id(), resolved_key)<br/>/usr/lib/python2.7/xmlrpclib.py:1243: in __call__<br/>    return self.__send(self.__name, args)<br/>/usr/lib/python2.7/xmlrpclib.py:1602: in __request<br/>    verbose=self.__verbose<br/>/usr/lib/python2.7/xmlrpclib.py:1283: in request<br/>    return self.single_request(host, handler, request_body, verbose)<br/>/usr/lib/python2.7/xmlrpclib.py:1311: in single_request<br/>    self.send_content(h, request_body)<br/>/usr/lib/python2.7/xmlrpclib.py:1459: in send_content<br/>    connection.endheaders(request_body)<br/>/usr/lib/python2.7/httplib.py:1053: in endheaders<br/>    self._send_output(message_body)<br/>/usr/lib/python2.7/httplib.py:897: in _send_output<br/>    self.send(msg)<br/>/usr/lib/python2.7/httplib.py:859: in send<br/>    self.connect()<br/>/usr/lib/python2.7/httplib.py:836: in connect<br/>    self.timeout, self.source_address)<br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/><br/>address = ('localhost', 11311), timeout = &lt;object object at 0x7f5b8b5b83b0&gt;, source_address = None<br/><br/>    def create_connection(address, timeout=_GLOBAL_DEFAULT_TIMEOUT,<br/>                          source_address=None):<br/>        """Connect to *address* and return the socket object.<br/>    <br/>        Convenience function.  Connect to *address* (a 2-tuple ``(host,<br/>        port)``) and return the socket object.  Passing the optional<br/>        *timeout* parameter will set the timeout on the socket instance<br/>        before attempting to connect.  If no *timeout* is supplied, the<br/>        global default timeout setting returned by :func:`getdefaulttimeout`<br/>        is used.  If *source_address* is set it must be a tuple of (host, port)<br/>        for the socket to bind as a source address before making the connection.<br/>        A host of '' or port 0 tells the OS to use the default.<br/>        """<br/>    <br/>        host, port = address<br/>        err = None<br/>        for res in getaddrinfo(host, port, 0, SOCK_STREAM):<br/>            af, socktype, proto, canonname, sa = res<br/>            sock = None<br/>            try:<br/>                sock = socket(af, socktype, proto)<br/>                if timeout is not _GLOBAL_DEFAULT_TIMEOUT:<br/>                    sock.settimeout(timeout)<br/>                if source_address:<br/>                    sock.bind(source_address)<br/>                sock.connect(sa)<br/>                return sock<br/>    <br/>            except error as _:<br/>                err = _<br/>                if sock is not None:<br/>                    sock.close()<br/>    <br/>        if err is not None:<br/>&gt;           raise err<br/><span class="error">E           error: [Errno 111] Connection refused</span><br/><br/>/usr/lib/python2.7/socket.py:575: error<br/>----------------------------- Captured stdout call -----------------------------<br/>['husky', 'locobot', 'sawyer', 'turtlebot']
</div></td></tr></tbody>
      <tbody class="failed results-table-row">
        <tr>
          <td class="col-result">Failed</td>
          <td class="col-name">tests/test_make_robots.py::test_bumper_topic_locobase_ilqr_none</td>
          <td class="col-duration">0.00</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">botname = 'locobot'<br/><br/>    def test_bumper_topic_locobase_ilqr_none(botname):<br/>&gt;       bot = test_make_robot_base(botname, 'ilqr', 'none')<br/><br/>test_make_robots.py:27: <br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/>test_make_robots.py:22: in test_make_robot_base<br/>    use_arm=False, use_camera=False,use_gripper=False)<br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/core.py:109: in __init__<br/>    self.base = base_class(self.configs, **base_config)<br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/locobot/base.py:204: in __init__<br/>    'use_base', False) or rospy.get_param(<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/client.py:465: in get_param<br/>    return _param_server[param_name] #MasterProxy does all the magic for us<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/msproxy.py:121: in __getitem__<br/>    code, msg, value = self.target.getParam(rospy.names.get_caller_id(), resolved_key)<br/>/usr/lib/python2.7/xmlrpclib.py:1243: in __call__<br/>    return self.__send(self.__name, args)<br/>/usr/lib/python2.7/xmlrpclib.py:1602: in __request<br/>    verbose=self.__verbose<br/>/usr/lib/python2.7/xmlrpclib.py:1283: in request<br/>    return self.single_request(host, handler, request_body, verbose)<br/>/usr/lib/python2.7/xmlrpclib.py:1311: in single_request<br/>    self.send_content(h, request_body)<br/>/usr/lib/python2.7/xmlrpclib.py:1459: in send_content<br/>    connection.endheaders(request_body)<br/>/usr/lib/python2.7/httplib.py:1053: in endheaders<br/>    self._send_output(message_body)<br/>/usr/lib/python2.7/httplib.py:897: in _send_output<br/>    self.send(msg)<br/>/usr/lib/python2.7/httplib.py:859: in send<br/>    self.connect()<br/>/usr/lib/python2.7/httplib.py:836: in connect<br/>    self.timeout, self.source_address)<br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/><br/>address = ('localhost', 11311), timeout = &lt;object object at 0x7f5b8b5b83b0&gt;, source_address = None<br/><br/>    def create_connection(address, timeout=_GLOBAL_DEFAULT_TIMEOUT,<br/>                          source_address=None):<br/>        """Connect to *address* and return the socket object.<br/>    <br/>        Convenience function.  Connect to *address* (a 2-tuple ``(host,<br/>        port)``) and return the socket object.  Passing the optional<br/>        *timeout* parameter will set the timeout on the socket instance<br/>        before attempting to connect.  If no *timeout* is supplied, the<br/>        global default timeout setting returned by :func:`getdefaulttimeout`<br/>        is used.  If *source_address* is set it must be a tuple of (host, port)<br/>        for the socket to bind as a source address before making the connection.<br/>        A host of '' or port 0 tells the OS to use the default.<br/>        """<br/>    <br/>        host, port = address<br/>        err = None<br/>        for res in getaddrinfo(host, port, 0, SOCK_STREAM):<br/>            af, socktype, proto, canonname, sa = res<br/>            sock = None<br/>            try:<br/>                sock = socket(af, socktype, proto)<br/>                if timeout is not _GLOBAL_DEFAULT_TIMEOUT:<br/>                    sock.settimeout(timeout)<br/>                if source_address:<br/>                    sock.bind(source_address)<br/>                sock.connect(sa)<br/>                return sock<br/>    <br/>            except error as _:<br/>                err = _<br/>                if sock is not None:<br/>                    sock.close()<br/>    <br/>        if err is not None:<br/>&gt;           raise err<br/><span class="error">E           error: [Errno 111] Connection refused</span><br/><br/>/usr/lib/python2.7/socket.py:575: error<br/>----------------------------- Captured stdout call -----------------------------<br/>['husky', 'locobot', 'sawyer', 'turtlebot']
</div></td></tr></tbody>
      <tbody class="failed results-table-row">
        <tr>
          <td class="col-result">Failed</td>
          <td class="col-name">tests/test_make_robots.py::test_make_robot_arm[True]</td>
          <td class="col-duration">0.04</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">botname = 'locobot', use_moveit = 'True'<br/><br/>    @pytest.mark.parametrize("use_moveit", ['True', 'False'])<br/>    def test_make_robot_arm(botname, use_moveit):<br/>        bot = Robot(botname, arm_config={'use_moveit': use_moveit},<br/>&gt;                   use_base=False)<br/><br/>test_make_robots.py:37: <br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/core.py:105: in __init__<br/>    self.arm = arm_class(self.configs, **arm_config)<br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/locobot/arm.py:40: in __init__<br/>    use_arm = rospy.get_param('use_arm', False)<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/client.py:465: in get_param<br/>    return _param_server[param_name] #MasterProxy does all the magic for us<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/msproxy.py:121: in __getitem__<br/>    code, msg, value = self.target.getParam(rospy.names.get_caller_id(), resolved_key)<br/>/usr/lib/python2.7/xmlrpclib.py:1243: in __call__<br/>    return self.__send(self.__name, args)<br/>/usr/lib/python2.7/xmlrpclib.py:1602: in __request<br/>    verbose=self.__verbose<br/>/usr/lib/python2.7/xmlrpclib.py:1283: in request<br/>    return self.single_request(host, handler, request_body, verbose)<br/>/usr/lib/python2.7/xmlrpclib.py:1311: in single_request<br/>    self.send_content(h, request_body)<br/>/usr/lib/python2.7/xmlrpclib.py:1459: in send_content<br/>    connection.endheaders(request_body)<br/>/usr/lib/python2.7/httplib.py:1053: in endheaders<br/>    self._send_output(message_body)<br/>/usr/lib/python2.7/httplib.py:897: in _send_output<br/>    self.send(msg)<br/>/usr/lib/python2.7/httplib.py:859: in send<br/>    self.connect()<br/>/usr/lib/python2.7/httplib.py:836: in connect<br/>    self.timeout, self.source_address)<br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/><br/>address = ('localhost', 11311), timeout = &lt;object object at 0x7f5b8b5b83b0&gt;, source_address = None<br/><br/>    def create_connection(address, timeout=_GLOBAL_DEFAULT_TIMEOUT,<br/>                          source_address=None):<br/>        """Connect to *address* and return the socket object.<br/>    <br/>        Convenience function.  Connect to *address* (a 2-tuple ``(host,<br/>        port)``) and return the socket object.  Passing the optional<br/>        *timeout* parameter will set the timeout on the socket instance<br/>        before attempting to connect.  If no *timeout* is supplied, the<br/>        global default timeout setting returned by :func:`getdefaulttimeout`<br/>        is used.  If *source_address* is set it must be a tuple of (host, port)<br/>        for the socket to bind as a source address before making the connection.<br/>        A host of '' or port 0 tells the OS to use the default.<br/>        """<br/>    <br/>        host, port = address<br/>        err = None<br/>        for res in getaddrinfo(host, port, 0, SOCK_STREAM):<br/>            af, socktype, proto, canonname, sa = res<br/>            sock = None<br/>            try:<br/>                sock = socket(af, socktype, proto)<br/>                if timeout is not _GLOBAL_DEFAULT_TIMEOUT:<br/>                    sock.settimeout(timeout)<br/>                if source_address:<br/>                    sock.bind(source_address)<br/>                sock.connect(sa)<br/>                return sock<br/>    <br/>            except error as _:<br/>                err = _<br/>                if sock is not None:<br/>                    sock.close()<br/>    <br/>        if err is not None:<br/>&gt;           raise err<br/><span class="error">E           error: [Errno 111] Connection refused</span><br/><br/>/usr/lib/python2.7/socket.py:575: error<br/>----------------------------- Captured stdout call -----------------------------<br/>['husky', 'locobot', 'sawyer', 'turtlebot']
</div></td></tr></tbody>
      <tbody class="failed results-table-row">
        <tr>
          <td class="col-result">Failed</td>
          <td class="col-name">tests/test_make_robots.py::test_make_robot_arm[False]</td>
          <td class="col-duration">0.00</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log">botname = 'locobot', use_moveit = 'False'<br/><br/>    @pytest.mark.parametrize("use_moveit", ['True', 'False'])<br/>    def test_make_robot_arm(botname, use_moveit):<br/>        bot = Robot(botname, arm_config={'use_moveit': use_moveit},<br/>&gt;                   use_base=False)<br/><br/>test_make_robots.py:37: <br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/core.py:105: in __init__<br/>    self.arm = arm_class(self.configs, **arm_config)<br/>../../pyenv_pyrobot/local/lib/python2.7/site-packages/pyrobot/locobot/arm.py:40: in __init__<br/>    use_arm = rospy.get_param('use_arm', False)<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/client.py:465: in get_param<br/>    return _param_server[param_name] #MasterProxy does all the magic for us<br/>/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/msproxy.py:121: in __getitem__<br/>    code, msg, value = self.target.getParam(rospy.names.get_caller_id(), resolved_key)<br/>/usr/lib/python2.7/xmlrpclib.py:1243: in __call__<br/>    return self.__send(self.__name, args)<br/>/usr/lib/python2.7/xmlrpclib.py:1602: in __request<br/>    verbose=self.__verbose<br/>/usr/lib/python2.7/xmlrpclib.py:1283: in request<br/>    return self.single_request(host, handler, request_body, verbose)<br/>/usr/lib/python2.7/xmlrpclib.py:1311: in single_request<br/>    self.send_content(h, request_body)<br/>/usr/lib/python2.7/xmlrpclib.py:1459: in send_content<br/>    connection.endheaders(request_body)<br/>/usr/lib/python2.7/httplib.py:1053: in endheaders<br/>    self._send_output(message_body)<br/>/usr/lib/python2.7/httplib.py:897: in _send_output<br/>    self.send(msg)<br/>/usr/lib/python2.7/httplib.py:859: in send<br/>    self.connect()<br/>/usr/lib/python2.7/httplib.py:836: in connect<br/>    self.timeout, self.source_address)<br/>_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ <br/><br/>address = ('localhost', 11311), timeout = &lt;object object at 0x7f5b8b5b83b0&gt;, source_address = None<br/><br/>    def create_connection(address, timeout=_GLOBAL_DEFAULT_TIMEOUT,<br/>                          source_address=None):<br/>        """Connect to *address* and return the socket object.<br/>    <br/>        Convenience function.  Connect to *address* (a 2-tuple ``(host,<br/>        port)``) and return the socket object.  Passing the optional<br/>        *timeout* parameter will set the timeout on the socket instance<br/>        before attempting to connect.  If no *timeout* is supplied, the<br/>        global default timeout setting returned by :func:`getdefaulttimeout`<br/>        is used.  If *source_address* is set it must be a tuple of (host, port)<br/>        for the socket to bind as a source address before making the connection.<br/>        A host of '' or port 0 tells the OS to use the default.<br/>        """<br/>    <br/>        host, port = address<br/>        err = None<br/>        for res in getaddrinfo(host, port, 0, SOCK_STREAM):<br/>            af, socktype, proto, canonname, sa = res<br/>            sock = None<br/>            try:<br/>                sock = socket(af, socktype, proto)<br/>                if timeout is not _GLOBAL_DEFAULT_TIMEOUT:<br/>                    sock.settimeout(timeout)<br/>                if source_address:<br/>                    sock.bind(source_address)<br/>                sock.connect(sa)<br/>                return sock<br/>    <br/>            except error as _:<br/>                err = _<br/>                if sock is not None:<br/>                    sock.close()<br/>    <br/>        if err is not None:<br/>&gt;           raise err<br/><span class="error">E           error: [Errno 111] Connection refused</span><br/><br/>/usr/lib/python2.7/socket.py:575: error<br/>----------------------------- Captured stdout call -----------------------------<br/>['husky', 'locobot', 'sawyer', 'turtlebot']
</div></td></tr></tbody></table></body></html>